<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.5.57">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Papers – DeMON</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<link href="./images/logobraindev.png" rel="icon" type="image/png">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-text-highlighting-styles">
<link href="site_libs/quarto-html/quarto-syntax-highlighting-dark.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-bootstrap" data-mode="light">
<link href="site_libs/bootstrap/bootstrap-dark.min.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-bootstrap" data-mode="dark">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>
<link href="site_libs/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet">
<script src="site_libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<link href="site_libs/vis-9.1.0/vis-network.min.css" rel="stylesheet">
<script src="site_libs/vis-9.1.0/vis-network.min.js"></script>
<script src="site_libs/visNetwork-binding-2.1.2/visNetwork.js"></script>
<script src="site_libs/core-js-2.5.3/shim.min.js"></script>
<script src="site_libs/react-18.2.0/react.min.js"></script>
<script src="site_libs/react-18.2.0/react-dom.min.js"></script>
<script src="site_libs/reactwidget-2.0.0/react-tools.js"></script>
<link href="site_libs/reactable-0.4.4/reactable.css" rel="stylesheet">
<script src="site_libs/reactable-binding-0.4.4/reactable.js"></script>


<link rel="stylesheet" href="styles.css">
</head>

<body class="nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a href="./index.html" class="navbar-brand navbar-brand-logo">
    <img src="./images/logobraindev.png" alt="" class="navbar-logo">
    </a>
    <a class="navbar-brand" href="./index.html">
    <span class="navbar-title">DeMON</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" role="menu" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="./index.html"> 
<span class="menu-text">Home</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./members.html"> 
<span class="menu-text">Members</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./projects.html"> 
<span class="menu-text">Projects</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link active" href="./papers.html" aria-current="page"> 
<span class="menu-text">Papers</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./takepart.html"> 
<span class="menu-text">Contact</span></a>
  </li>  
</ul>
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
</ul>
          </div> <!-- /navcollapse -->
            <div class="quarto-navbar-tools">
  <a href="" class="quarto-color-scheme-toggle quarto-navigation-tool  px-1" onclick="window.quartoToggleColorScheme(); return false;" title="Toggle dark mode"><i class="bi"></i></a>
</div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-full page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar zindex-bottom">
        
    </div>
<!-- main -->
<main class="content column-page" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Papers</h1>
</div>



<div class="quarto-title-meta column-page">

    
  
    
  </div>
  


</header>


<div>

</div>
<div class="cell quarto-layout-panel" data-layout-ncol="2">
<div class="quarto-layout-row">
<div class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: center;">
<p><img src="papers_files/figure-html/wc-1.png" class="img-fluid" width="672"></p>
</div>
<div class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<div class="visNetwork html-widget html-fill-item" id="htmlwidget-6c29e7e4525a4ce9d2e3" style="width:100%;height:464px;"></div>
<script type="application/json" data-for="htmlwidget-6c29e7e4525a4ce9d2e3">{"x":{"nodes":{"name":["Personalised regional modelling predicts tau progression in the human brain.","Accelerating biomedical discoveries in brain health through transformative neuropathology of aging and neurodegeneration.","The Global Neurodegeneration Proteomics Consortium: biomarker and drug target discovery for common neurodegenerative diseases and aging.","Age and Alzheimer's disease affect functional connectivity along separate axes of functional brain organization.","Benchmarking the AI-based diagnostic potential of plasma proteomics for neurodegenerative disease in 17,170 people.","A novel approach to resilience and its links with education and Alzheimer's disease genetics.","Machine learning prediction of tau-PET in Alzheimer's disease using plasma, MRI, and clinical data.","Disparate and shared transcriptomic signatures associated with cortical atrophy in genetic behavioral variant frontotemporal degeneration.","Integrating amyloid and tau imaging with proteomics and genomics in Alzheimer's disease.","Proteomic changes in Alzheimer's disease associated with progressive Aβ plaque and tau tangle pathologies.","Identification of distinct and shared biomarker panels in different manifestations of cerebral small vessel disease through proteomic profiling.","Disease progression modelling reveals heterogeneity in trajectories of Lewy-type α-synuclein pathology.","Deciphering the functional specialization of whole-brain spatiomolecular gradients in the adult brain.","A machine learning-based prediction of tau load and distribution in Alzheimer's disease using plasma, MRI and clinical variables.","A generalizable data-driven model of atrophy heterogeneity and progression in memory clinic settings.","Novel data-driven subtypes and stages of brain atrophy in the ALS-FTD spectrum.","The molecular genetic landscape of human brain size variation.","Connectome-based modelling of neurodegenerative diseases: towards precision medicine and mechanistic insight.","Comparison of Group-Level and Individualized Brain Regions for Measuring Change in Longitudinal Tau Positron Emission Tomography in Alzheimer Disease.","Data-driven neuropathological staging and subtyping of TDP-43 proteinopathies.","Local molecular and global connectomic contributions to cross-disorder cortical abnormalities.","Differentiating amyloid beta spread in autosomal dominant and sporadic Alzheimer's disease.","Subtypes of Alzheimer's disease: questions, controversy, and meaning.","Ordinal SuStaIn: Subtype and Stage Inference for Clinical Scores, Visual Ratings, and Other Ordinal Data.","Four distinct trajectories of tau deposition identified in Alzheimer's disease.","Mapping gene transcription and neurocognition across human neocortex.","Early stages of tau pathology and its associations with functional connectivity, atrophy and memory.","Spread of pathological tau proteins through communicating neurons in human Alzheimer's disease.","Morphometric network differences in ageing versus Alzheimer's disease dementia.","Cerebrospinal fluid reference proteins increase accuracy and interpretability of biomarkers for brain diseases.","Issues and recommendations for the residual approach to quantifying cognitive resilience and reserve.","A molecular gradient along the longitudinal axis of the human hippocampus informs large-scale behavioral systems."],"group":[2,2,2,4,2,3,2,4,2,4,4,1,4,2,1,1,4,4,2,1,4,3,3,1,1,4,1,2,3,3,3,4],"color":["steelblue","steelblue","steelblue","goldenrod","steelblue","forestgreen","steelblue","goldenrod","steelblue","goldenrod","goldenrod","tomato","goldenrod","steelblue","tomato","tomato","goldenrod","goldenrod","steelblue","tomato","goldenrod","forestgreen","forestgreen","tomato","tomato","goldenrod","tomato","steelblue","forestgreen","forestgreen","forestgreen","goldenrod"],"id":[1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32],"title":["Personalised regional modelling predicts tau progression in the human brain.","Accelerating biomedical discoveries in brain health through transformative neuropathology of aging and neurodegeneration.","The Global Neurodegeneration Proteomics Consortium: biomarker and drug target discovery for common neurodegenerative diseases and aging.","Age and Alzheimer's disease affect functional connectivity along separate axes of functional brain organization.","Benchmarking the AI-based diagnostic potential of plasma proteomics for neurodegenerative disease in 17,170 people.","A novel approach to resilience and its links with education and Alzheimer's disease genetics.","Machine learning prediction of tau-PET in Alzheimer's disease using plasma, MRI, and clinical data.","Disparate and shared transcriptomic signatures associated with cortical atrophy in genetic behavioral variant frontotemporal degeneration.","Integrating amyloid and tau imaging with proteomics and genomics in Alzheimer's disease.","Proteomic changes in Alzheimer's disease associated with progressive Aβ plaque and tau tangle pathologies.","Identification of distinct and shared biomarker panels in different manifestations of cerebral small vessel disease through proteomic profiling.","Disease progression modelling reveals heterogeneity in trajectories of Lewy-type α-synuclein pathology.","Deciphering the functional specialization of whole-brain spatiomolecular gradients in the adult brain.","A machine learning-based prediction of tau load and distribution in Alzheimer's disease using plasma, MRI and clinical variables.","A generalizable data-driven model of atrophy heterogeneity and progression in memory clinic settings.","Novel data-driven subtypes and stages of brain atrophy in the ALS-FTD spectrum.","The molecular genetic landscape of human brain size variation.","Connectome-based modelling of neurodegenerative diseases: towards precision medicine and mechanistic insight.","Comparison of Group-Level and Individualized Brain Regions for Measuring Change in Longitudinal Tau Positron Emission Tomography in Alzheimer Disease.","Data-driven neuropathological staging and subtyping of TDP-43 proteinopathies.","Local molecular and global connectomic contributions to cross-disorder cortical abnormalities.","Differentiating amyloid beta spread in autosomal dominant and sporadic Alzheimer's disease.","Subtypes of Alzheimer's disease: questions, controversy, and meaning.","Ordinal SuStaIn: Subtype and Stage Inference for Clinical Scores, Visual Ratings, and Other Ordinal Data.","Four distinct trajectories of tau deposition identified in Alzheimer's disease.","Mapping gene transcription and neurocognition across human neocortex.","Early stages of tau pathology and its associations with functional connectivity, atrophy and memory.","Spread of pathological tau proteins through communicating neurons in human Alzheimer's disease.","Morphometric network differences in ageing versus Alzheimer's disease dementia.","Cerebrospinal fluid reference proteins increase accuracy and interpretability of biomarkers for brain diseases.","Issues and recommendations for the residual approach to quantifying cognitive resilience and reserve.","A molecular gradient along the longitudinal axis of the human hippocampus informs large-scale behavioral systems."]},"edges":{"from":[1,1,2,3,4,4,4,4,5,5,5,6,7,7,8,9,9,10,11,11,12,12,13,13,14,14,15,15,16,16,16,17,18,18,18,18,18,19,19,20,20,21,22,23,24,25,26,27],"to":[28,7,25,9,18,27,29,10,10,19,30,31,25,23,16,10,30,30,10,30,24,20,32,26,25,23,27,25,20,25,24,25,28,21,25,22,23,25,23,25,24,26,28,25,25,28,32,28],"arrows":["to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to"],"smooth":[true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true],"shadow":[true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true]},"nodesToDataframe":true,"edgesToDataframe":true,"options":{"width":"100%","height":"100%","nodes":{"shape":"dot"},"manipulation":{"enabled":false}},"groups":["2","4","3","1"],"width":null,"height":null,"idselection":{"enabled":false},"byselection":{"enabled":false},"main":null,"submain":null,"footer":null,"background":"rgba(0, 0, 0, 0)"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</div>
<div class="cell">
<div class="cell-output-display">
<div class="reactable html-widget html-fill-item" id="htmlwidget-bf2d8ab2087f70535b0f" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-bf2d8ab2087f70535b0f">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"Title":["Retinal tau phosphorylation in Alzheimer's disease: A mass spectrometry study.","Two-step detection of Lewy body pathology via smell-function testing and CSF α-synuclein seed amplification.","The Global Neurodegeneration Proteomics Consortium: biomarker and drug target discovery for common neurodegenerative diseases and aging.","Accelerating biomedical discoveries in brain health through transformative neuropathology of aging and neurodegeneration.","Personalised regional modelling predicts tau progression in the human brain.","Benchmarking the AI-based diagnostic potential of plasma proteomics for neurodegenerative disease in 17,170 people.","A novel approach to resilience and its links with education and Alzheimer's disease genetics.","Age and Alzheimer's disease affect functional connectivity along separate axes of functional brain organization.","Disparate and shared transcriptomic signatures associated with cortical atrophy in genetic behavioral variant frontotemporal degeneration.","Machine learning prediction of tau-PET in Alzheimer's disease using plasma, MRI, and clinical data.","Integration across biophysical scales identifies molecular and cellular correlates of person-to-person variability in human brain connectivity.","Proteomic changes in Alzheimer's disease associated with progressive Aβ plaque and tau tangle pathologies.","A machine learning-based prediction of tau load and distribution in Alzheimer's disease using plasma, MRI and clinical variables.","Integrating amyloid and tau imaging with proteomics and genomics in Alzheimer's disease.","Disparate and shared transcriptomic signatures associated with cortical atrophy in genetic bvFTD.","A generalizable data-driven model of atrophy heterogeneity and progression in memory clinic settings.","Deciphering the functional specialization of whole-brain spatiomolecular gradients in the adult brain.","Disease progression modelling reveals heterogeneity in trajectories of Lewy-type α-synuclein pathology.","Identification of distinct and shared biomarker panels in different manifestations of cerebral small vessel disease through proteomic profiling.","Cerebrospinal fluid reference proteins increase accuracy and interpretability of biomarkers for brain diseases.","Novel data-driven subtypes and stages of brain atrophy in the ALS-FTD spectrum.","The molecular genetic landscape of human brain size variation.","Functional imaging studies of acute administration of classic psychedelics, ketamine, and MDMA: Methodological limitations and convergent results.","Connectome-based modelling of neurodegenerative diseases: towards precision medicine and mechanistic insight.","Data-driven neuropathological staging and subtyping of TDP-43 proteinopathies.","Comparison of Group-Level and Individualized Brain Regions for Measuring Change in Longitudinal Tau Positron Emission Tomography in Alzheimer Disease.","Local molecular and global connectomic contributions to cross-disorder cortical abnormalities.","Issues and recommendations for the residual approach to quantifying cognitive resilience and reserve.","Subtypes of Alzheimer's disease: questions, controversy, and meaning.","Differentiating amyloid beta spread in autosomal dominant and sporadic Alzheimer's disease.","Early stages of tau pathology and its associations with functional connectivity, atrophy and memory.","Mapping gene transcription and neurocognition across human neocortex.","Four distinct trajectories of tau deposition identified in Alzheimer's disease.","Vascular risk factors are associated with a decline in resting-state functional connectivity in cognitively unimpaired individuals at risk for Alzheimer's disease: Vascular risk factors and functional connectivity changes.","Ordinal SuStaIn: Subtype and Stage Inference for Clinical Scores, Visual Ratings, and Other Ordinal Data.","Latent atrophy factors related to phenotypical variants of posterior cortical atrophy.","Spread of pathological tau proteins through communicating neurons in human Alzheimer's disease.","A molecular gradient along the longitudinal axis of the human hippocampus informs large-scale behavioral systems.","Morphometric network differences in ageing versus Alzheimer's disease dementia."],"Auth":["Santiago J,  Pocevičiūtė D ...  Wennström M","Mastenbroek SE,  Collij LE ...  Hansson O","Imam F,  Saloner R ...  Lovestone S","Murray ME,  Smith C ...  Phatnani H","Chaggar P,  Vogel JW ...  Goriely A","An L,  Pichet-Binette A ...  Vogel JW","Carrigan M,  Bocancea DI ...  Ossenkoppele R","Rittmo J,  Franzmeier N ...  Vogel JW","Shen T,  Vogel JW ...  McMillan CT","Karlsson L,  Vogel J ...  Hansson O","Ng B,  Tasaki S ...  Herskowitz JH","Pichet Binette A,  Gaiteri C ...  Hansson O","Karlsson L,  Vogel J ...  Hansson O","Vilkaite G,  Vogel J ...  Mattsson-Carlgren N","Shen T,  Vogel JW ...  McMillan CT","Baumeister H,  Vogel JW ...  Berron D","Vogel JW,  Alexander-Bloch AF ...  Seidlitz J","Mastenbroek SE,  Vogel JW ...  Hansson O","Hristovska I,  Binette AP ...  Hansson O","Karlsson L,  Vogel J ...  Hansson O","Shen T,  Vogel JW ...  McMillan CT","Seidlitz J,  Mallard TT ...  Alexander-Bloch AF","Linguiti S,  Vogel JW ...  Satterthwaite TD","Vogel JW,  Corriveau-Lecavalier N ...  Ewers M","Young AL,  Vogel JW ...  Hansson O","Leuzy A,  Binette AP ...  Hansson O","Hansen JY,  Shafiei G ...  Misic B","Elman JA,  Vogel JW ...  Kremen WS","Vogel JW,  Hansson O ...  Hansson O","Levitis E,  Vogel JW ...  Evans AC","Berron D,  Vogel JW ...  Hansson O","Hansen JY,  Markello RD ...  Misic B","Vogel JW,  Young AL ...  Hansson O","Köbe T,  Binette AP ...  Villeneuve S","Young AL,  Vogel JW ...  Alexander DC","Groot C,  Yeo BTT ...  Ossenkoppele R","Vogel JW,  Iturria-Medina Y ...  Hansson O","Vogel JW,  La Joie R ...  Evans AC","Pichet Binette A,  Gonneaud J ...  Vachon-Presseau E"],"Journal":["Neurobiology of disease","Nature communications","Nature medicine","Neuron","PLoS biology","medRxiv : the preprint server for health sciences","Alzheimer's & dementia : the journal of the Alzheimer's Association","bioRxiv : the preprint server for biology","Molecular neurodegeneration","Alzheimer's & dementia : the journal of the Alzheimer's Association","Nature neuroscience","Nature neuroscience","medRxiv : the preprint server for health sciences","Cell reports. Medicine","medRxiv : the preprint server for health sciences","Brain : a journal of neurology","Proceedings of the National Academy of Sciences of the United States of America","Nature communications","medRxiv : the preprint server for health sciences","Nature communications","Translational neurodegeneration","Cell reports","Neuroscience and biobehavioral reviews","Nature reviews. Neuroscience","Brain : a journal of neurology","JAMA neurology","Nature communications","Alzheimer's research & therapy","Trends in neurosciences","Brain communications","Brain : a journal of neurology","Nature human behaviour","Nature medicine","NeuroImage","Frontiers in artificial intelligence","Neurology","Nature communications","Nature communications","Brain : a journal of neurology"],"Year":[2025,2025,2025,2025,2025,2025,2025,2025,2025,2025,2024,2024,2024,2024,2024,2024,2024,2024,2024,2024,2023,2023,2023,2023,2023,2023,2022,2022,2022,2022,2021,2021,2021,2021,2021,2020,2020,2020,2020]},"columns":[{"id":".details","name":"","type":null,"style":{"fontSize":"11px"},"sortable":false,"resizable":false,"filterable":false,"searchable":false,"width":45,"align":"center","details":[{"name":"Reactable","attribs":{"data":{".rownames":["Author","Abstract"],"V1":["Santiago J, Pocevičiūtė D, Vogel J, Brinkmalm G, Wennström M","Most neurodegenerative diseases, including Alzheimer's disease (AD) and multiple sclerosis (MS), feature abnormal tau phosphorylation (p-tau) in the brain. Prior immunostaining studies have shown p-tau accumulation in the AD retina, suggesting it may mirror brain tau pathology.We used mass spectrometry to quantify p-tau peptides in matched retinal and hippocampal samples from non-demented controls (NC, n = 8), AD (n = 12), and MS (n = 4). We compared p-tau levels across diagnoses and analysed correlations between retinal p-tau variants, hippocampal p-tau, and neuropathological changes.Tau peptides phosphorylated at T181, S199/S202, T231, T231 + T235, S396 + T403/S404, and T403/S404 were detected in retinas. Total tau phosphorylation and phosphorylation at S199/S202 and T231 were significantly higher in AD cases compared to NC. These two, along with p-tau S396 + T403/S404, were also higher in cases with high amyloid-beta (Aβ) Braak stages compared to those with low Aβ Braak stages. Higher Aβ stages were also correlated with higher peak intensities of p-tau S199/S202 and S396 + T403/S404, and retinal p-tau S396 + T403/S404 and T403/S404 correlated with neurofibrillary tangle (NFT) Braak stages. Additionally, p-tau S396 + T403/S404 in the retina was associated with corresponding phosphorylation in the hippocampus.Our findings reveal both overlapping and distinct p-tau patterns in retina and hippocampus, with a notable link for p-tau S396 + T403/S404. This enhances our understanding of tauopathies in both tissues and supports retinal tau as a promising biomarker for AD diagnosis and monitoring.Copyright © 2025 The Authors. Published by Elsevier Inc. All rights reserved."]},"columns":[{"id":".rownames","name":"","type":"character","sortable":false,"filterable":false,"rowHeader":true},{"id":"V1","name":"V1","type":"character","width":600}],"highlight":true,"outlined":true,"dataKey":"7c1c08340e277f907df81448865e4f05","static":false,"nested":true},"children":[]},{"name":"Reactable","attribs":{"data":{".rownames":["Author","Abstract"],"V1":["Mastenbroek SE, Collij LE, Vogel JW, Caldera S, Serrano GE, Adler CH, Vargiu CM, Palmqvist S, Barkhof F, Parchi P, Beach TG, Ossenkoppele R, Hansson O","Cerebrospinal fluid (CSF) α-synuclein (α-syn) seed amplification assays (SAAs) can detect Lewy body pathology (LBP) with high accuracy but are invasive and costly. To address these challenges, this study evaluated a two-step workflow combining prescreening via smell-function testing with confirmatory CSF α-syn SAA testing only in individuals with reduced smell, for predicting postmortem LBP status. Among 358 autopsied participants, the two-step workflow predicted brain LBP with high accuracy overall (94%), and within clinical subgroups (clinical parkinsonism=95%; clinical Alzheimer's disease [AD]=94%; clinically unimpaired [CU]=93%). It reduced the need for confirmatory CSF testing by 43% overall (23% clinical parkinsonism; 35% clinical AD; 80% CU). In an independent in vivo cohort (N=1209), the workflow predicted CSF α-syn SAA status with 79% accuracy and reduced CSF testing by 26%. This approach may reduce invasive CSF testing, alleviating patient burden and lowering healthcare costs.© 2025. The Author(s)."]},"columns":[{"id":".rownames","name":"","type":"character","sortable":false,"filterable":false,"rowHeader":true},{"id":"V1","name":"V1","type":"character","width":600}],"highlight":true,"outlined":true,"dataKey":"ba44b83b891ab54217ac6d730f708ac9","static":false,"nested":true},"children":[]},{"name":"Reactable","attribs":{"data":{".rownames":["Author","Abstract"],"V1":["Imam F, Saloner R, Vogel JW, Krish V, Abdel-Azim G, Ali M, An L, Anastasi F, Bennett D, Pichet Binette A, Boxer AL, Bringmann M, Burns JM, Cruchaga C, Dage JL, Farinas A, Ferrucci L, Finney CA, Frasier M, Hansson O, Hohman TJ, Johnson ECB, Kivimaki M, Korologou-Linden R, Ruiz Laza A, Levey AI, Liepelt-Scarfone I, Lu L, Mattsson-Carlgren N, Middleton LT, Nho K, Oh HS, Petersen RC, Reiman EM, Robinson O, Rothstein JD, Saykin AJ, Shvetcov A, Slawson C, Smets B, Suárez-Calvet M, Tijms BM, Timmers M, Vieira F, Vilor-Tejedor N, Visser PJ, Walker KA, Winchester LM, Wyss-Coray T, Yang C, Bose N, Lovestone S","More than 57 million people globally suffer from neurodegenerative diseases, a figure expected to double every 20 years. Despite this growing burden, there are currently no cures, and treatment options remain limited due to disease heterogeneity, prolonged preclinical and prodromal phases, poor understanding of disease mechanisms, and diagnostic challenges. Identifying novel biomarkers is crucial for improving early detection, prognosis, staging and subtyping of these conditions. High-dimensional molecular studies in biofluids ('omics') offer promise for scalable biomarker discovery, but challenges in assembling large, diverse datasets hinder progress. To address this, the Global Neurodegeneration Proteomics Consortium (GNPC)-a public-private partnership-established one of the world's largest harmonized proteomic datasets. It includes approximately 250 million unique protein measurements from multiple platforms from more than 35,000 biofluid samples (plasma, serum and cerebrospinal fluid) contributed by 23 partners, alongside associated clinical data spanning Alzheimer's disease (AD), Parkinson's disease (PD), frontotemporal dementia (FTD) and amyotrophic lateral sclerosis (ALS). This dataset is accessible to GNPC members via the Alzheimer's Disease Data Initiative's AD Workbench, a secure cloud-based environment, and will be available to the wider research community on 15 July 2025. Here we present summary analyses of the plasma proteome revealing disease-specific differential protein abundance and transdiagnostic proteomic signatures of clinical severity. Furthermore, we describe a robust plasma proteomic signature of APOE ε4 carriership, reproducible across AD, PD, FTD and ALS, as well as distinct patterns of organ aging across these conditions. This work demonstrates the power of international collaboration, data sharing and open science to accelerate discovery in neurodegeneration research.© 2025. The Author(s)."]},"columns":[{"id":".rownames","name":"","type":"character","sortable":false,"filterable":false,"rowHeader":true},{"id":"V1","name":"V1","type":"character","width":600}],"highlight":true,"outlined":true,"dataKey":"1e3bc3d39ed489ea2acdf7a34ffd786e","static":false,"nested":true},"children":[]},{"name":"Reactable","attribs":{"data":{".rownames":["Author","Abstract"],"V1":["Murray ME, Smith C, Menon V, Keene CD, Lein E, Hawrylycz M, Aguzzi A, Benedetti B, Brose K, Caetano-Anolles K, Sillero MIC, Crary JF, De Jager PL, Faustin A, Flanagan ME, Gokce O, Grant SGN, Grinberg LT, Gutman DA, Hillman EMC, Huang Z, Irwin DJ, Jones DT, Kapasi A, Karch CM, Kukull WT, Lashley T, Lee EB, Lehner T, Parkkinen L, Pedersen M, Pritchett D, Rutledge MH, Schneider JA, Seeley WW, Shepherd CE, Spires-Jones TL, Steen JA, Sutherland M, Vickovic S, Zhang B, Stewart DJ, Keiser MJ, Vogel JW, Dugger BN, Phatnani H","Transformative neuropathology is redefining human brain research by integrating foundational descriptive pathology with advanced methodologies. These approaches, spanning multi-omics studies and machine learning applications, will drive discovery for the identification of biomarkers, therapeutic targets, and complex disease patterns through comprehensive analyses of postmortem human brain tissue. Yet critical challenges remain, including the sustainability of brain banks, expanding donor participation, strengthening training pipelines, enabling rapid autopsies, supporting collaborative platforms, and integrating data across modalities. Innovations in digital pathology, tissue quality enhancement, harmonization of data standards, and machine learning integration offer opportunities to accelerate tissue-level \"pathomics\" research in brain health through cross-disciplinary collaborations. Lessons from neuroimaging, particularly in establishing common data frameworks and multi-site collaborations, offer a valuable roadmap for streamlining innovations. In this perspective, we outline actionable solutions for leveraging existing resources and strengthening collaboration -where we envision future opportunities to drive translational discoveries stemming from transformative neuropathology.Copyright © 2025 The Author(s). Published by Elsevier Inc. All rights reserved."]},"columns":[{"id":".rownames","name":"","type":"character","sortable":false,"filterable":false,"rowHeader":true},{"id":"V1","name":"V1","type":"character","width":600}],"highlight":true,"outlined":true,"dataKey":"406ffc7ce463b6d3255fbd2c29155ca7","static":false,"nested":true},"children":[]},{"name":"Reactable","attribs":{"data":{".rownames":["Author","Abstract"],"V1":["Chaggar P, Vogel JW, Binette AP, Thompson TB, Strandberg O, Mattsson-Carlgren N, Karlsson L, Stomrud E, Jbabdi S, Magon S, Klein G, Hansson O, Goriely A","Aggregation of the hyperphosphorylated tau protein is a central driver of Alzheimer's disease, and its accumulation exhibits a rich spatiotemporal pattern that unfolds during the course of the disease, sequentially progressing through the brain across axonal connections. It is unclear how this spatiotemporal process is orchestrated, namely, to what extent the spread of pathologic tau is governed by transport between brain regions, local production, or both. To address this, we develop a mechanistic model from tau PET data to describe tau dynamics along the Alzheimer's disease timeline. Our analysis reveals longitudinal changes in production and transport dynamics in two independent cohorts, with subjects in the early stage of the disease exhibiting transport-dominated spread, consistent with an initial spread of pathological tau seeds, and subjects in the late stage disease characterized primarily by local tau production. Further, we demonstrate that the model can predict accurately subject-specific longitudinal tau accumulation at the regional level, potentially providing a new clinical tool to monitor and classify patient disease progression.Copyright: © 2025 Chaggar et al. This is an open access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited."]},"columns":[{"id":".rownames","name":"","type":"character","sortable":false,"filterable":false,"rowHeader":true},{"id":"V1","name":"V1","type":"character","width":600}],"highlight":true,"outlined":true,"dataKey":"dd443da04cb58c81b07be19b03715ea7","static":false,"nested":true},"children":[]},{"name":"Reactable","attribs":{"data":{".rownames":["Author","Abstract"],"V1":["An L, Pichet-Binette A, Hristovska I, Vilkaite G, Yu X, Smets B, Saloner R, Tasaki S, Xu Y, Krish V, Imam F, Janelidze S, van Westen D, Stomrud E, Whelan CD, Palmqvist S, Ossenkoppele R, Mattsson-Carlgren N, Hansson O, Vogel JW","Co-pathology is a common feature of neurodegenerative diseases that complicates diagnosis, treatment and clinical management. However, sensitive, specific and scalable biomarkers for in vivo pathological diagnosis are not available for most neurodegenerative neuropathologies. Here, we present ProtAIDe-Dx, a deep joint-learning model trained on 17,170 patients and controls that uses plasma proteomics to provide simultaneous probabilistic diagnosis across six conditions associated with dementia in aging. ProtAIDe-Dx achieves cross-validated balanced classification accuracy of 69%-96% and AUCs > 79% across all conditions. The model's diagnostic probabilities highlighted subgroups of patients with co-pathologies, and were associated with pathology-specific biomarkers in an external sample, even among cognitively unimpaired people. Model interpretation revealed a suite of protein networks marking shared and specific biological processes across diseases, and identified novel and previously described proteins discriminating each diagnosis. ProtAIDe-Dx significantly improved biomarker-based differential diagnosis in a memory clinic sample, pinpointing proteins leading to diagnostic decisions at an individual level. Together, this work highlights the promise of plasma proteomics to improve patient-level diagnostic work-up with a single blood draw."]},"columns":[{"id":".rownames","name":"","type":"character","sortable":false,"filterable":false,"rowHeader":true},{"id":"V1","name":"V1","type":"character","width":600}],"highlight":true,"outlined":true,"dataKey":"1df145e774112c2bec2c42db6aa2656d","static":false,"nested":true},"children":[]},{"name":"Reactable","attribs":{"data":{".rownames":["Author","Abstract"],"V1":["Carrigan M, Bocancea DI, Vogel J, van Loenhoud AC, Tesi N, Barkhof F, Lucassen PJ, van der Flier WM, Krugers HJ, Van der Lee SJ, Ossenkoppele R","Cognitive resilience refers to maintaining cognitive function despite Alzheimer's disease (AD) pathophysiology.We analyzed amyloid-positive individuals across clinical stages of AD in two cohorts: the Amsterdam Dementia Cohort (ADC, N = 1036) and Alzheimer's Disease Neuroimaging Initiative (ADNI, N = 685). Cognitive resilience was conceptualized from a canonical correlation analysis of magnetic resonance imaging and neuropsychological data in each cohort separately. Model validation involved education as a resilience proxy and key genetic factors (apolipoprotein E [APOE] ε4 and APOE ε2) of AD. We explored associations between 83 AD risk loci and cognitive resilience.Resilience was correlated with education (ADC: β = 0.144, p < 0.001; ADNI: β = 0.149, p < 0.001) and APOE ε4 (βmeta-analysis = -0.052, p = 0.014). Exploratory single nucleotide polymorphism meta-analysis identified potential involvement of genetic variants around genes UNC5CL, USP6NL, and TPCN1 in lower, and genes COX7C and MINDY2 in higher resilience.Our novel resilience approach showed conceptual validity and potential for future discovery of resilience-related genetic variants.·We define a novel approach to resilience using canonical correlation analysis (CCA). ·Apolipoprotein E ε4 is linked to lower resilience, suggesting increased vulnerability. ·Genetic loci around COX7C and MINDY2 are potentially involved in higher resilience. ·This novel approach may be used for multi-cohort studies such as genome-wide association studies in the future.© 2025 The Author(s). Alzheimer's & Dementia published by Wiley Periodicals LLC on behalf of Alzheimer's Association."]},"columns":[{"id":".rownames","name":"","type":"character","sortable":false,"filterable":false,"rowHeader":true},{"id":"V1","name":"V1","type":"character","width":600}],"highlight":true,"outlined":true,"dataKey":"987a2e409a805c1bb941cfcf776c362e","static":false,"nested":true},"children":[]},{"name":"Reactable","attribs":{"data":{".rownames":["Author","Abstract"],"V1":["Rittmo J, Franzmeier N, Strandberg O, Chauveau L, Satterthwaite TD, Wisse LE, Spotorno N, Behjat HH, Dehsarvi A, van Westen D, Anijärv TE, Palmqvist S, Janelidze S, Stomrud E, Ossenkoppele R, Mattsson-Carlgren N, Hansson O, Vogel JW","Aging and Alzheimer's disease (AD) are associated with alterations in functional connectivity (FC), yet their spatial and temporal characteristics remain debated. Whole-cortex functional gradients, which organize regions along axes of functional similarity, are positioned here as a framework for understanding such alterations. Across two independent cohorts (BioFINDER-2, N=973; ADNI, N=129), we demonstrate that hyper- and hypoconnectivity in both healthy aging and biomarker-confirmed AD progression coexist systematically, forming consistent spatial patterns that align with two distinct axes of brain organization. Using a combination of longitudinal and non-linear analyses, we show that the early (but not late) stages of AD pathology accumulation is associated with functional alteration along the sensory-association axis, a pattern that vanishes in later stages of AD. However, functional alteration along the sensory-association axis is associated with worse cognition throughout the AD spectrum, and even in older adults without AD pathology, suggesting that these FC patterns may reflect a general neural response to cognitive strain. Independently of AD, older age was associated with alterations instead along the executive-nonexecutive axis, a finding that was consistent throughout the adult lifespan. These findings highlight the fundamental role of intrinsic functional organization in shaping how the brain responds to aging and to AD, helping to resolve previously reported discrepancies."]},"columns":[{"id":".rownames","name":"","type":"character","sortable":false,"filterable":false,"rowHeader":true},{"id":"V1","name":"V1","type":"character","width":600}],"highlight":true,"outlined":true,"dataKey":"bec6446925d19167e421ac0a152e829b","static":false,"nested":true},"children":[]},{"name":"Reactable","attribs":{"data":{".rownames":["Author","Abstract"],"V1":["Shen T, Vogel JW, Van Deerlin VM, Suh E, Dratch L, Phillips JS, Massimo L, Lee EB, Irwin DJ, McMillan CT","Cortical atrophy is a common manifestation in behavioral variant frontotemporal degeneration (bvFTD), exhibiting spatial heterogeneity across various genetic subgroups, which may be driven by distinct biological mechanisms.We employed an integrative imaging transcriptomics approach to identify both disparate and shared transcriptomic signatures associated with cortical thickness in bvFTD with C9orf72 repeat expansions or pathogenic variants in GRN or MAPT. Functional enrichment analyses were conducted on each gene list significantly associated with cortical thickness. Additionally, we mapped neurotransmitter receptor/transporter density maps to the cortical thickness maps, to uncover different correlation patterns for each genetic form. Furthermore, we examined whether the identified genes were enriched for pathology-related genes by using previously identified genes linked to TDP-43 positive neurons and genes associated with tau pathology.For each genetic form of bvFTD, we identified cortical thickness signatures and gene sets associated with them. The cortical thickness associated genes for GRN-bvFTD were significantly involved in neurotransmitter system and circadian entrainment. The different patterns of spatial correlations between synaptic density and cortical thinning, further confirmed the critical role of neurotransmission and synaptic signaling in shaping brain structure, especially in the GRN-bvFTD group. Furthermore, we observed significant overlap between genes linked to TDP-43 pathology and the gene sets associated with cortical thickness in C9orf72-bvFTD and GRN-bvFTD but not the MAPT-bvFTD group providing specificity for our associations. C9orf72-bvFTD and GRN-bvFTD also shared genes displaying consistent directionality, with those exhibiting either positive or negative correlations with cortical thickness in C9orf72-bvFTD showing the same direction (positive or negative) in GRN-bvFTD. MAPT-bvFTD displayed more pronounced differences in transcriptomic signatures compared to the other two genetic forms. The genes that exhibited significantly positive or negative correlations with cortical thickness in MAPT-bvFTD showed opposing directionality in C9orf72-bvFTD and GRN-bvFTD.Overall, this integrative transcriptomic approach identified several new shared and disparate genes associated with regional vulnerability with increased biological interpretation including overlap with synaptic density maps and pathologically-specific gene expression. These findings illuminated the intricate molecular underpinnings contributing to the heterogeneous nature of disease distribution in bvFTD with distinct genetic backgrounds.© 2025. The Author(s)."]},"columns":[{"id":".rownames","name":"","type":"character","sortable":false,"filterable":false,"rowHeader":true},{"id":"V1","name":"V1","type":"character","width":600}],"highlight":true,"outlined":true,"dataKey":"30fd4fdaccd7b44d359253a880e52f15","static":false,"nested":true},"children":[]},{"name":"Reactable","attribs":{"data":{".rownames":["Author","Abstract"],"V1":["Karlsson L, Vogel J, Arvidsson I, Åström K, Strandberg O, Seidlitz J, Bethlehem RAI, Stomrud E, Ossenkoppele R, Ashton NJ, Zetterberg H, Blennow K, Palmqvist S, Smith R, Janelidze S, La Joie R, Rabinovici GD, Binette AP, Mattsson-Carlgren N, Hansson O","Tau positron emission tomography (PET) is a reliable neuroimaging technique for assessing regional load of tau pathology in the brain, but its routine clinical use is limited by cost and accessibility barriers.We thoroughly investigated the ability of various machine learning models to predict clinically useful tau-PET composites (load and laterality index) from low-cost and non-invasive features, for example, clinical variables, plasma biomarkers, and structural magnetic resonance imaging (MRI).Models including plasma biomarkers yielded the most accurate predictions of tau-PET burden (best model: R-squared = 0.66-0.69), with especially high contribution from plasma phosphorylated tau-217 (p-tau217). MRI variables were the best predictors of asymmetric tau load between the two hemispheres (best model: R-squared = 0.28-0.42). The models showed high generalizability to external test cohorts with data collected at multiple sites. Through a proof-of-concept two-step classification workflow, we also demonstrated possible model translations to a clinical setting.This study highlights the promising and limiting aspects of using machine learning to predict tau-PET from scalable cost-effective variables, with findings relevant for clinical settings and future research.Accessible variables showed potential in estimating tau tangle load and distribution. Plasma phosphorylated tau-217 (p-tau217) and magnetic resonance imaging (MRI) were the best predictors of different tau-PET (positron emission tomography) composites. Machine learning models demonstrated high generalizability across AD cohorts.© 2025 The Author(s). Alzheimer's & Dementia published by Wiley Periodicals LLC on behalf of Alzheimer's Association."]},"columns":[{"id":".rownames","name":"","type":"character","sortable":false,"filterable":false,"rowHeader":true},{"id":"V1","name":"V1","type":"character","width":600}],"highlight":true,"outlined":true,"dataKey":"af2ef8835c37be9d7fd86c96e6e39819","static":false,"nested":true},"children":[]},{"name":"Reactable","attribs":{"data":{".rownames":["Author","Abstract"],"V1":["Ng B, Tasaki S, Greathouse KM, Walker CK, Zhang A, Covitz S, Cieslak M, Weber AJ, Adamson AB, Andrade JP, Poovey EH, Curtis KA, Muhammad HM, Seidlitz J, Satterthwaite T, Bennett DA, Seyfried NT, Vogel J, Gaiteri C, Herskowitz JH","Brain connectivity arises from interactions across biophysical scales, ranging from molecular to cellular to anatomical to network level. To date, there has been little progress toward integrated analysis across these scales. To bridge this gap, from a unique cohort of 98 individuals, we collected antemortem neuroimaging and genetic data, as well as postmortem dendritic spine morphometric, proteomic and gene expression data from the superior frontal and inferior temporal gyri. Through the integration of the molecular and dendritic spine morphology data, we identified hundreds of proteins that explain interindividual differences in functional connectivity and structural covariation. These proteins are enriched for synaptic structures and functions, energy metabolism and RNA processing. By integrating data at the genetic, molecular, subcellular and tissue levels, we link specific biochemical changes at synapses to connectivity between brain regions. These results demonstrate the feasibility of integrating data from vastly different biophysical scales to provide a more comprehensive understanding of brain connectivity.© 2024. The Author(s)."]},"columns":[{"id":".rownames","name":"","type":"character","sortable":false,"filterable":false,"rowHeader":true},{"id":"V1","name":"V1","type":"character","width":600}],"highlight":true,"outlined":true,"dataKey":"cdc0fe5a5d096d2aca9a83921cf5031b","static":false,"nested":true},"children":[]},{"name":"Reactable","attribs":{"data":{".rownames":["Author","Abstract"],"V1":["Pichet Binette A, Gaiteri C, Wennström M, Kumar A, Hristovska I, Spotorno N, Salvadó G, Strandberg O, Mathys H, Tsai LH, Palmqvist S, Mattsson-Carlgren N, Janelidze S, Stomrud E, Vogel JW, Hansson O","Proteomics can shed light on the dynamic and multifaceted alterations in neurodegenerative disorders like Alzheimer's disease (AD). Combining radioligands measuring β-amyloid (Aβ) plaques and tau tangles with cerebrospinal fluid proteomics, we uncover molecular events mirroring different stages of AD pathology in living humans. We found 127 differentially abundant proteins (DAPs) across the AD spectrum. The strongest Aβ-related proteins were mainly expressed in glial cells and included SMOC1 and ITGAM. A dozen proteins linked to ATP metabolism and preferentially expressed in neurons were independently associated with tau tangle load and tau accumulation. Only 20% of the DAPs were also altered in other neurodegenerative diseases, underscoring AD's distinct proteome. Two co-expression modules related, respectively, to protein metabolism and microglial immune response encompassed most DAPs, with opposing, staggered trajectories along the AD continuum. We unveil protein signatures associated with Aβ and tau proteinopathy in vivo, offering insights into complex neural responses and potential biomarkers and therapeutics targeting different disease stages.© 2024. The Author(s)."]},"columns":[{"id":".rownames","name":"","type":"character","sortable":false,"filterable":false,"rowHeader":true},{"id":"V1","name":"V1","type":"character","width":600}],"highlight":true,"outlined":true,"dataKey":"15375b4a4a88948db6f81504feddd08b","static":false,"nested":true},"children":[]},{"name":"Reactable","attribs":{"data":{".rownames":["Author","Abstract"],"V1":["Karlsson L, Vogel J, Arvidsson I, Åström K, Strandberg O, Seidlitz J, Bethlehem RAI, Stomrud E, Ossenkoppele R, Ashton NJ, Zetterberg H, Blennow K, Palmqvist S, Smith R, Janelidze S, Joie R, Rabinovici GD, Binette AP, Mattsson-Carlgren N, Hansson O","Tau positron emission tomography (PET) is a reliable neuroimaging technique for assessing regional load of tau pathology in the brain, commonly used in Alzheimer's disease (AD) research and clinical trials. However, its routine clinical use is limited by cost and accessibility barriers. Here we explore using machine learning (ML) models to predict clinically useful tau-PET composites from low-cost and non-invasive features, e.g., basic clinical variables, plasma biomarkers, and structural magnetic resonance imaging (MRI). Results demonstrated that models including plasma biomarkers yielded the most accurate predictions of tau-PET burden (best model: R-squared=0.66-0.68), with especially high contribution from plasma P-tau217. In contrast, MRI variables stood out as best predictors (best model: R-squared=0.28-0.42) of asymmetric tau load between the two hemispheres (an example of clinically relevant spatial information). The models showed high generalizability to external test cohorts with data collected at multiple sites. Based on these results, we also propose a proof-of-concept two-step classification workflow, demonstrating how the ML models can be translated to a clinical setting. This study uncovers current potential in predicting tau-PET information from scalable cost-effective variables, which could improve diagnosis and prognosis of AD."]},"columns":[{"id":".rownames","name":"","type":"character","sortable":false,"filterable":false,"rowHeader":true},{"id":"V1","name":"V1","type":"character","width":600}],"highlight":true,"outlined":true,"dataKey":"5392ff6f7be670c99b71a94abe364f58","static":false,"nested":true},"children":[]},{"name":"Reactable","attribs":{"data":{".rownames":["Author","Abstract"],"V1":["Vilkaite G, Vogel J, Mattsson-Carlgren N","Alzheimer's disease (AD) is the most common neurodegenerative disease and is characterized by the aggregation of β-amyloid (Aβ) and tau in the brain. Breakthroughs in disease-modifying treatments targeting Aβ bring new hope for the management of AD. But to effectively modify and someday even prevent AD, a better understanding is needed of the biological mechanisms that underlie and link Aβ and tau in AD. Developments of high-throughput omics, including genomics, proteomics, and transcriptomics, together with molecular imaging of Aβ and tau with positron emission tomography (PET), allow us to discover and understand the biological pathways that regulate the aggregation and spread of Aβ and tau in living humans. The field of integrated omics and PET studies of Aβ and tau in AD is growing rapidly. We here provide an update of this field, both in terms of biological insights and in terms of future clinical implications of integrated omics-molecular imaging studies.Copyright © 2024 The Author(s). Published by Elsevier Inc. All rights reserved."]},"columns":[{"id":".rownames","name":"","type":"character","sortable":false,"filterable":false,"rowHeader":true},{"id":"V1","name":"V1","type":"character","width":600}],"highlight":true,"outlined":true,"dataKey":"10827a3fdfe3331b0f47ec32600905be","static":false,"nested":true},"children":[]},{"name":"Reactable","attribs":{"data":{".rownames":["Author","Abstract"],"V1":["Shen T, Vogel JW, Van Deerlin VM, Suh E, Dratch L, Phillips JS, Massimo L, Lee EB, Irwin DJ, McMillan CT","Cortical atrophy in behavioral variant frontotemporal degeneration (bvFTD) exhibits spatial heterogeneity across genetic subgroups, potentially driven by distinct biological mechanisms. Using an integrative imaging-transcriptomics approach, we identified disparate and shared transcriptomic signatures associated with cortical thickness in C9orf72 , GRN or MAPT -related bvFTD. Genes associated with cortical thinning in GRN -bvFTD were implicated in neurotransmission, further supported by mapping synaptic density maps to cortical thickness maps. Previously identified genes linked to TDP-43 positive neurons were significantly overlapped with genes associated with C9orf72 -bvFTD and GRN -bvFTD, but not MAPT -bvFTD providing specificity for our associations. C9orf72 -bvFTD and GRN -bvFTD shared genes displaying consistent directionality of correlations with cortical thickness, while MAPT -bvFTD displayed more pronounced differences in transcriptomic signatures with opposing directionality. Overall, we identified disparate and shared genes tied to regional vulnerability with increased biological interpretation including overlap with synaptic density maps and pathologically-specific gene expression, illuminating intricate molecular underpinnings contributing to heterogeneities in bvFTD."]},"columns":[{"id":".rownames","name":"","type":"character","sortable":false,"filterable":false,"rowHeader":true},{"id":"V1","name":"V1","type":"character","width":600}],"highlight":true,"outlined":true,"dataKey":"49f17d5950e23d471d0e1d525f5d01f1","static":false,"nested":true},"children":[]},{"name":"Reactable","attribs":{"data":{".rownames":["Author","Abstract"],"V1":["Baumeister H, Vogel JW, Insel PS, Kleineidam L, Wolfsgruber S, Stark M, Gellersen HM, Yakupov R, Schmid MC, Lüsebrink F, Brosseron F, Ziegler G, Freiesleben SD, Preis L, Schneider LS, Spruth EJ, Altenstein S, Lohse A, Fliessbach K, Vogt IR, Bartels C, Schott BH, Rostamzadeh A, Glanz W, Incesoy EI, Butryn M, Janowitz D, Rauchmann BS, Kilimann I, Goerss D, Munk MH, Hetzer S, Dechent P, Ewers M, Scheffler K, Wuestefeld A, Strandberg O, van Westen D, Mattsson-Carlgren N, Janelidze S, Stomrud E, Palmqvist S, Spottke A, Laske C, Teipel S, Perneczky R, Buerger K, Schneider A, Priller J, Peters O, Ramirez A, Wiltfang J, Heneka MT, Wagner M, Düzel E, Jessen F, Hansson O, Berron D","Memory clinic patients are a heterogeneous population representing various aetiologies of pathological ageing. It is not known whether divergent spatiotemporal progression patterns of brain atrophy, as previously described in Alzheimer's disease patients, are prevalent and clinically meaningful in this group of older adults. To uncover distinct atrophy subtypes, we applied the Subtype and Stage Inference (SuStaIn) algorithm to baseline structural MRI data from 813 participants enrolled in the DELCODE cohort (mean ± standard deviation, age = 70.67 ± 6.07 years, 52% females). Participants were cognitively unimpaired (n = 285) or fulfilled diagnostic criteria for subjective cognitive decline (n = 342), mild cognitive impairment (n = 118) or dementia of the Alzheimer's type (n = 68). Atrophy subtypes were compared in baseline demographics, fluid Alzheimer's disease biomarker levels, the Preclinical Alzheimer Cognitive Composite (PACC-5) as well as episodic memory and executive functioning. PACC-5 trajectories over up to 240 weeks were examined. To test whether baseline atrophy subtype and stage predicted clinical trajectories before manifest cognitive impairment, we analysed PACC-5 trajectories and mild cognitive impairment conversion rates of cognitively unimpaired participants and those with subjective cognitive decline. Limbic-predominant and hippocampal-sparing atrophy subtypes were identified. Limbic-predominant atrophy initially affected the medial temporal lobes, followed by further temporal regions and, finally, the remaining cortical regions. At baseline, this subtype was related to older age, more pathological Alzheimer's disease biomarker levels, APOE ε4 carriership and an amnestic cognitive impairment. Hippocampal-sparing atrophy initially occurred outside the temporal lobe, with the medial temporal lobe spared up to advanced atrophy stages. This atrophy pattern also affected individuals with positive Alzheimer's disease biomarkers and was associated with more generalized cognitive impairment. Limbic-predominant atrophy, in all participants and in only unimpaired participants, was linked to more negative longitudinal PACC-5 slopes than observed in participants without or with hippocampal-sparing atrophy and increased the risk of mild cognitive impairment conversion. SuStaIn modelling was repeated in a sample from the Swedish BioFINDER-2 cohort. Highly similar atrophy progression patterns and associated cognitive profiles were identified. Cross-cohort model generalizability, at both the subject and the group level, was excellent, indicating reliable performance in previously unseen data. The proposed model is a promising tool for capturing heterogeneity among older adults at early at-risk states for Alzheimer's disease in applied settings. The implementation of atrophy subtype- and stage-specific end points might increase the statistical power of pharmacological trials targeting early Alzheimer's disease.© The Author(s) 2024. Published by Oxford University Press on behalf of the Guarantors of Brain."]},"columns":[{"id":".rownames","name":"","type":"character","sortable":false,"filterable":false,"rowHeader":true},{"id":"V1","name":"V1","type":"character","width":600}],"highlight":true,"outlined":true,"dataKey":"b2054bf2d74bcf1e3bb5d94d7b3c6ec9","static":false,"nested":true},"children":[]},{"name":"Reactable","attribs":{"data":{".rownames":["Author","Abstract"],"V1":["Vogel JW, Alexander-Bloch AF, Wagstyl K, Bertolero MA, Markello RD, Pines A, Sydnor VJ, Diaz-Papkovich A, Hansen JY, Evans AC, Bernhardt B, Misic B, Satterthwaite TD, Seidlitz J","Cortical arealization arises during neurodevelopment from the confluence of molecular gradients representing patterned expression of morphogens and transcription factors. However, whether similar gradients are maintained in the adult brain remains unknown. Here, we uncover three axes of topographic variation in gene expression in the adult human brain that specifically capture previously identified rostral-caudal, dorsal-ventral, and medial-lateral axes of early developmental patterning. The interaction of these spatiomolecular gradients i) accurately reconstructs the position of brain tissue samples, ii) delineates known functional territories, and iii) can model the topographical variation of diverse cortical features. The spatiomolecular gradients are distinct from canonical cortical axes differentiating the primary sensory cortex from the association cortex, but radiate in parallel with the axes traversed by local field potentials along the cortex. We replicate all three molecular gradients in three independent human datasets as well as two nonhuman primate datasets and find that each gradient shows a distinct developmental trajectory across the lifespan. The gradients are composed of several well-known transcription factors (e.g., PAX6 and SIX3), and a small set of genes shared across gradients are strongly enriched for multiple diseases. Together, these results provide insight into the developmental sculpting of functionally distinct brain regions, governed by three robust transcriptomic axes embedded within brain parenchyma."]},"columns":[{"id":".rownames","name":"","type":"character","sortable":false,"filterable":false,"rowHeader":true},{"id":"V1","name":"V1","type":"character","width":600}],"highlight":true,"outlined":true,"dataKey":"698a0b071367266b775abcf393641157","static":false,"nested":true},"children":[]},{"name":"Reactable","attribs":{"data":{".rownames":["Author","Abstract"],"V1":["Mastenbroek SE, Vogel JW, Collij LE, Serrano GE, Tremblay C, Young AL, Arce RA, Shill HA, Driver-Dunckley ED, Mehta SH, Belden CM, Atri A, Choudhury P, Barkhof F, Adler CH, Ossenkoppele R, Beach TG, Hansson O","Lewy body (LB) diseases, characterized by the aggregation of misfolded α-synuclein proteins, exhibit notable clinical heterogeneity. This may be due to variations in accumulation patterns of LB neuropathology. Here we apply a data-driven disease progression model to regional neuropathological LB density scores from 814 brain donors with Lewy pathology. We describe three inferred trajectories of LB pathology that are characterized by differing clinicopathological presentation and longitudinal antemortem clinical progression. Most donors (81.9%) show earliest pathology in the olfactory bulb, followed by accumulation in either limbic (60.8%) or brainstem (21.1%) regions. The remaining donors (18.1%) initially exhibit abnormalities in brainstem regions. Early limbic pathology is associated with Alzheimer's disease-associated characteristics while early brainstem pathology is associated with progressive motor impairment and substantial LB pathology outside of the brain. Our data provides evidence for heterogeneity in the temporal spread of LB pathology, possibly explaining some of the clinical disparities observed in Lewy body disease.© 2024. The Author(s)."]},"columns":[{"id":".rownames","name":"","type":"character","sortable":false,"filterable":false,"rowHeader":true},{"id":"V1","name":"V1","type":"character","width":600}],"highlight":true,"outlined":true,"dataKey":"8017b7fccd7424a3ba9ccccfff8ef140","static":false,"nested":true},"children":[]},{"name":"Reactable","attribs":{"data":{".rownames":["Author","Abstract"],"V1":["Hristovska I, Binette AP, Kumar A, Gaiteri C, Karlsson L, Strandberg O, Janelidze S, van Westen D, Stomrud E, Palmqvist S, Ossenkoppele R, Mattsson-Carlgren N, Vogel JW, Hansson O","The pathophysiology underlying various manifestations of cerebral small vessel disease (cSVD) remains obscure. Using cerebrospinal fluid proximity extension assays and co-expression network analysis of 2,943 proteins, we found common and distinct proteomic signatures between white matter lesions (WML), microbleeds and infarcts measured in 856 living patients, and validated WML-associated proteins in three additional datasets. Proteins indicative of extracellular matrix dysregulation and vascular remodeling, including ELN, POSTN, CCN2 and MMP12 were elevated across all cSVD manifestations, with MMP12 emerging as an early cSVD indicator. cSVD-associated proteins formed a co-abundance network linked to metabolism and enriched in endothelial and arterial smooth muscle cells, showing elevated levels at early disease manifestations. Later disease stages involved changes in microglial proteins, associated with longitudinal WML progression, and changes in neuronal proteins mediating WML-associated cognitive decline. These findings provide an atlas of novel cSVD biomarkers and a promising roadmap for the next generation of cSVD therapeutics."]},"columns":[{"id":".rownames","name":"","type":"character","sortable":false,"filterable":false,"rowHeader":true},{"id":"V1","name":"V1","type":"character","width":600}],"highlight":true,"outlined":true,"dataKey":"86f67e651bacc357e9b4f4dd6ca68e4b","static":false,"nested":true},"children":[]},{"name":"Reactable","attribs":{"data":{".rownames":["Author","Abstract"],"V1":["Karlsson L, Vogel J, Arvidsson I, Åström K, Janelidze S, Blennow K, Palmqvist S, Stomrud E, Mattsson-Carlgren N, Hansson O","Cerebrospinal fluid (CSF) biomarkers reflect brain pathophysiology and are used extensively in translational research as well as in clinical practice for diagnosis of neurological diseases, e.g., Alzheimer's disease (AD). However, CSF biomarker concentrations may be influenced by non-disease related inter-individual variability. Here we use a data-driven approach to demonstrate the existence of inter-individual variability in mean standardized CSF protein levels. We show that these non-disease related differences cause many commonly reported CSF biomarkers to be highly correlated, thereby producing misleading results if not accounted for. To adjust for this inter-individual variability, we identified and evaluated high-performing reference proteins which improved the diagnostic accuracy of key CSF AD biomarkers. Our reference protein method attenuates the risk for false positive findings, and improves the sensitivity and specificity of CSF biomarkers, with broad implications for both research and clinical practice.© 2024. The Author(s)."]},"columns":[{"id":".rownames","name":"","type":"character","sortable":false,"filterable":false,"rowHeader":true},{"id":"V1","name":"V1","type":"character","width":600}],"highlight":true,"outlined":true,"dataKey":"98ae895558e9c394a5a03b3bf31a7a97","static":false,"nested":true},"children":[]},{"name":"Reactable","attribs":{"data":{".rownames":["Author","Abstract"],"V1":["Shen T, Vogel JW, Duda J, Phillips JS, Cook PA, Gee J, Elman L, Quinn C, Amado DA, Baer M, Massimo L, Grossman M, Irwin DJ, McMillan CT","TDP-43 proteinopathies represent a spectrum of neurological disorders, anchored clinically on either end by amyotrophic lateral sclerosis (ALS) and frontotemporal degeneration (FTD). The ALS-FTD spectrum exhibits a diverse range of clinical presentations with overlapping phenotypes, highlighting its heterogeneity. This study was aimed to use disease progression modeling to identify novel data-driven spatial and temporal subtypes of brain atrophy and its progression in the ALS-FTD spectrum.We used a data-driven procedure to identify 13 anatomic clusters of brain volume for 57 behavioral variant FTD (bvFTD; with either autopsy-confirmed TDP-43 or TDP-43 proteinopathy-associated genetic variants), 103 ALS, and 47 ALS-FTD patients with likely TDP-43. A Subtype and Stage Inference (SuStaIn) model was trained to identify subtypes of individuals along the ALS-FTD spectrum with distinct brain atrophy patterns, and we related subtypes and stages to clinical, genetic, and neuropathological features of disease.SuStaIn identified three novel subtypes: two disease subtypes with predominant brain atrophy in either prefrontal/somatomotor regions or limbic-related regions, and a normal-appearing group without obvious brain atrophy. The limbic-predominant subtype tended to present with more impaired cognition, higher frequencies of pathogenic variants in TBK1 and TARDBP genes, and a higher proportion of TDP-43 types B, E and C. In contrast, the prefrontal/somatomotor-predominant subtype had higher frequencies of pathogenic variants in C9orf72 and GRN genes and higher proportion of TDP-43 type A. The normal-appearing brain group showed higher frequency of ALS relative to ALS-FTD and bvFTD patients, higher cognitive capacity, higher proportion of lower motor neuron onset, milder motor symptoms, and lower frequencies of genetic pathogenic variants. The overall SuStaIn stages also correlated with evidence for clinical progression including longer disease duration, higher King's stage, and cognitive decline. Additionally, SuStaIn stages differed across clinical phenotypes, genotypes and types of TDP-43 pathology.Our findings suggest distinct neurodegenerative subtypes of disease along the ALS-FTD spectrum that can be identified in vivo, each with distinct brain atrophy, clinical, genetic and pathological patterns.© 2023. The Author(s)."]},"columns":[{"id":".rownames","name":"","type":"character","sortable":false,"filterable":false,"rowHeader":true},{"id":"V1","name":"V1","type":"character","width":600}],"highlight":true,"outlined":true,"dataKey":"7e4f20b093b98c65688cfd297bfb866d","static":false,"nested":true},"children":[]},{"name":"Reactable","attribs":{"data":{".rownames":["Author","Abstract"],"V1":["Seidlitz J, Mallard TT, Vogel JW, Lee YH, Warrier V, Ball G, Hansson O, Hernandez LM, Mandal AS, Wagstyl K, Lombardo MV, Courchesne E, Glessner JT, Satterthwaite TD, Bethlehem RAI, Bernstock JD, Tasaki S, Ng B, Gaiteri C, Smoller JW, Ge T, Gur RE, Gandal MJ, Alexander-Bloch AF","Human brain size changes dynamically through early development, peaks in adolescence, and varies up to 2-fold among adults. However, the molecular genetic underpinnings of interindividual variation in brain size remain unknown. Here, we leveraged postmortem brain RNA sequencing and measurements of brain weight (BW) in 2,531 individuals across three independent datasets to identify 928 genome-wide significant associations with BW. Genes associated with higher or lower BW showed distinct neurodevelopmental trajectories and spatial patterns that mapped onto functional and cellular axes of brain organization. Expression of BW genes was predictive of interspecies differences in brain size, and bioinformatic annotation revealed enrichment for neurogenesis and cell-cell communication. Genome-wide, transcriptome-wide, and phenome-wide association analyses linked BW gene sets to neuroimaging measurements of brain size and brain-related clinical traits. Cumulatively, these results represent a major step toward delineating the molecular pathways underlying human brain size variation in health and disease.Copyright © 2023 The Authors. Published by Elsevier Inc. All rights reserved."]},"columns":[{"id":".rownames","name":"","type":"character","sortable":false,"filterable":false,"rowHeader":true},{"id":"V1","name":"V1","type":"character","width":600}],"highlight":true,"outlined":true,"dataKey":"395835b957eba9ff981665ea4e45509c","static":false,"nested":true},"children":[]},{"name":"Reactable","attribs":{"data":{".rownames":["Author","Abstract"],"V1":["Linguiti S, Vogel JW, Sydnor VJ, Pines A, Wellman N, Basbaum A, Eickhoff CR, Eickhoff SB, Edwards RR, Larsen B, McKinstry-Wu A, Scott JC, Roalf DR, Sharma V, Strain EC, Corder G, Dworkin RH, Satterthwaite TD","Functional magnetic resonance imaging (fMRI) is increasingly used to non-invasively study the acute impact of psychedelics on the human brain. While fMRI is a promising tool for measuring brain function in response to psychedelics, it also has known methodological challenges. We conducted a systematic review of fMRI studies examining acute responses to experimentally administered psychedelics in order to identify convergent findings and characterize heterogeneity in the literature. We reviewed 91 full-text papers; these studies were notable for substantial heterogeneity in design, task, dosage, drug timing, and statistical approach. Data recycling was common, with 51 unique samples across 91 studies. Fifty-seven studies (54%) did not meet contemporary standards for Type I error correction or control of motion artifact. Psilocybin and LSD were consistently reported to moderate the connectivity architecture of the sensorimotor-association cortical axis. Studies also consistently reported that ketamine administration increased activation in the dorsomedial prefrontal cortex. Moving forward, use of best practices such as pre-registration, standardized image processing and statistical testing, and data sharing will be important in this rapidly developing field.Copyright © 2023 Elsevier Ltd. All rights reserved."]},"columns":[{"id":".rownames","name":"","type":"character","sortable":false,"filterable":false,"rowHeader":true},{"id":"V1","name":"V1","type":"character","width":600}],"highlight":true,"outlined":true,"dataKey":"331e52095020385c5bdbf00f91cf7c43","static":false,"nested":true},"children":[]},{"name":"Reactable","attribs":{"data":{".rownames":["Author","Abstract"],"V1":["Vogel JW, Corriveau-Lecavalier N, Franzmeier N, Pereira JB, Brown JA, Maass A, Botha H, Seeley WW, Bassett DS, Jones DT, Ewers M","Neurodegenerative diseases are the most common cause of dementia. Although their underlying molecular pathologies have been identified, there is substantial heterogeneity in the patterns of progressive brain alterations across and within these diseases. Recent advances in neuroimaging methods have revealed that pathological proteins accumulate along specific macroscale brain networks, implicating the network architecture of the brain in the system-level pathophysiology of neurodegenerative diseases. However, the extent to which 'network-based neurodegeneration' applies across the wide range of neurodegenerative disorders remains unclear. Here, we discuss the state-of-the-art of neuroimaging-based connectomics for the mapping and prediction of neurodegenerative processes. We review findings supporting brain networks as passive conduits through which pathological proteins spread. As an alternative view, we also discuss complementary work suggesting that network alterations actively modulate the spreading of pathological proteins between connected brain regions. We conclude this Perspective by proposing an integrative framework in which connectome-based models can be advanced along three dimensions of innovation: incorporating parameters that modulate propagation behaviour on the basis of measurable biological features; building patient-tailored models that use individual-level information and allowing model parameters to interact dynamically over time. We discuss promises and pitfalls of these strategies for improving disease insights and moving towards precision medicine.© 2023. Springer Nature Limited."]},"columns":[{"id":".rownames","name":"","type":"character","sortable":false,"filterable":false,"rowHeader":true},{"id":"V1","name":"V1","type":"character","width":600}],"highlight":true,"outlined":true,"dataKey":"3eee9fe1c99c87a2ec139329fb12a0cf","static":false,"nested":true},"children":[]},{"name":"Reactable","attribs":{"data":{".rownames":["Author","Abstract"],"V1":["Young AL, Vogel JW, Robinson JL, McMillan CT, Ossenkoppele R, Wolk DA, Irwin DJ, Elman L, Grossman M, Lee VMY, Lee EB, Hansson O","TAR DNA-binding protein-43 (TDP-43) accumulation is the primary pathology underlying several neurodegenerative diseases. Charting the progression and heterogeneity of TDP-43 accumulation is necessary to better characterize TDP-43 proteinopathies, but current TDP-43 staging systems are heuristic and assume each syndrome is homogeneous. Here, we use data-driven disease progression modelling to derive a fine-grained empirical staging system for the classification and differentiation of frontotemporal lobar degeneration due to TDP-43 (FTLD-TDP, n = 126), amyotrophic lateral sclerosis (ALS, n = 141) and limbic-predominant age-related TDP-43 encephalopathy neuropathologic change (LATE-NC) with and without Alzheimer's disease (n = 304). The data-driven staging of ALS and FTLD-TDP complement and extend previously described human-defined staging schema for ALS and behavioural variant frontotemporal dementia. In LATE-NC individuals, progression along data-driven stages was positively associated with age, but negatively associated with age in individuals with FTLD-TDP. Using only regional TDP-43 severity, our data driven model distinguished individuals diagnosed with ALS, FTLD-TDP or LATE-NC with a cross-validated accuracy of 85.9%, with misclassifications associated with mixed pathological diagnosis, age and genetic mutations. Adding age and SuStaIn stage to this model increased accuracy to 92.3%. Our model differentiates LATE-NC from FTLD-TDP, though some overlap was observed between late-stage LATE-NC and early-stage FTLD-TDP. We further tested for the presence of subtypes with distinct regional TDP-43 progression patterns within each diagnostic group, identifying two distinct cortical-predominant and brainstem-predominant subtypes within FTLD-TDP and a further two subcortical-predominant and corticolimbic-predominant subtypes within ALS. The FTLD-TDP subtypes exhibited differing proportions of TDP-43 type, while there was a trend for age differing between ALS subtypes. Interestingly, a negative relationship between age and SuStaIn stage was seen in the brainstem/subcortical-predominant subtype of each proteinopathy. No subtypes were observed for the LATE-NC group, despite aggregating individuals with and without Alzheimer's disease and a larger sample size for this group. Overall, we provide an empirical pathological TDP-43 staging system for ALS, FTLD-TDP and LATE-NC, which yielded accurate classification. We further demonstrate that there is substantial heterogeneity amongst ALS and FTLD-TDP progression patterns that warrants further investigation in larger cross-cohort studies.© The Author(s) 2023. Published by Oxford University Press on behalf of the Guarantors of Brain."]},"columns":[{"id":".rownames","name":"","type":"character","sortable":false,"filterable":false,"rowHeader":true},{"id":"V1","name":"V1","type":"character","width":600}],"highlight":true,"outlined":true,"dataKey":"e5795eddb29dc5d6a952ec2bdda7b9e9","static":false,"nested":true},"children":[]},{"name":"Reactable","attribs":{"data":{".rownames":["Author","Abstract"],"V1":["Leuzy A, Binette AP, Vogel JW, Klein G, Borroni E, Tonietto M, Strandberg O, Mattsson-Carlgren N, Palmqvist S, Pontecorvo MJ, Iaccarino L, Stomrud E, Ossenkoppele R, Smith R, Hansson O","Longitudinal tau positron emission tomography (PET) is a relevant outcome in clinical trials evaluating disease-modifying therapies in Alzheimer disease (AD). A key unanswered question is whether the use of participant-specific (individualized) regions of interest (ROIs) is superior to conventional approaches where the same ROI (group-level) is used for each participant.To compare group- and participant-level ROIs in participants at different stages of the AD clinical continuum in terms of annual percentage change in tau-PET standardized uptake value ratio (SUVR) and sample size requirements.This was a longitudinal cohort study with consecutive participant enrollment between September 18, 2017, and November 15, 2021. Included in the analysis were participants with mild cognitive impairment and AD dementia from the prospective and longitudinal Swedish Biomarkers For Identifying Neurodegenerative Disorders Early and Reliably 2 (BioFINDER-2) study; in addition, a validation sample (the AVID 05e, Expedition-3, Alzheimer's Disease Neuroimaging Initiative [ADNI], and BioFINDER-1 study cohorts) was also included.Tau PET (BioFINDER-2, [18F]RO948; validation sample, [18F]flortaucipir), 7 group-level (5 data-driven stages, meta-temporal, whole brain), and 5 individualized ROIs.Annual percentage change in tau-PET SUVR across ROIs. Sample size requirements in simulated clinical trials using tau PET as an outcome were also calculated.A total of 215 participants (mean [SD] age, 71.4 (7.5) years; 111 male [51.6%]) from the BioFINDER-2 study were included in this analysis: 97 amyloid-β (Aβ)-positive cognitively unimpaired (CU) individuals, 77 with Aβ-positive mild cognitive impairment (MCI), and 41 with AD dementia. In the validation sample were 137 Aβ-positive CU participants, 144 with Aβ-positive MCI, and 125 with AD dementia. Mean (SD) follow-up time was 1.8 (0.3) years. Using group-level ROIs, the largest annual percentage increase in tau-PET SUVR in Aβ-positive CU individuals was seen in a composite ROI combining the entorhinal cortex, hippocampus, and amygdala (4.29%; 95% CI, 3.42%-5.16%). In individuals with Aβ-positive MCI, the greatest change was seen in the temporal cortical regions (5.82%; 95% CI, 4.67%-6.97%), whereas in those with AD dementia, the greatest change was seen in the parietal regions (5.22%; 95% CI, 3.95%-6.49%). Significantly higher estimates of annual percentage change were found using several of the participant-specific ROIs. Importantly, the simplest participant-specific approach, where change in tau PET was calculated in an ROI that best matched the participant's data-driven disease stage, performed best in all 3 subgroups. For the power analysis, sample size reductions for the participant-specific ROIs ranged from 15.94% (95% CI, 8.14%-23.74%) to 72.10% (95% CI, 67.10%-77.20%) compared with the best-performing group-level ROIs. Findings were replicated using [18F]flortaucipir.Finding suggest that certain individualized ROIs carry an advantage over group-level ROIs for assessing longitudinal tau changes and increase the power to detect treatment effects in AD clinical trials using longitudinal tau PET as an outcome."]},"columns":[{"id":".rownames","name":"","type":"character","sortable":false,"filterable":false,"rowHeader":true},{"id":"V1","name":"V1","type":"character","width":600}],"highlight":true,"outlined":true,"dataKey":"cbbb7d9e6663bcbe7f01e4119f35cde0","static":false,"nested":true},"children":[]},{"name":"Reactable","attribs":{"data":{".rownames":["Author","Abstract"],"V1":["Hansen JY, Shafiei G, Vogel JW, Smart K, Bearden CE, Hoogman M, Franke B, van Rooij D, Buitelaar J, McDonald CR, Sisodiya SM, Schmaal L, Veltman DJ, van den Heuvel OA, Stein DJ, van Erp TGM, Ching CRK, Andreassen OA, Hajek T, Opel N, Modinos G, Aleman A, van der Werf Y, Jahanshad N, Thomopoulos SI, Thompson PM, Carson RE, Dagher A, Misic B","Numerous brain disorders demonstrate structural brain abnormalities, which are thought to arise from molecular perturbations or connectome miswiring. The unique and shared contributions of these molecular and connectomic vulnerabilities to brain disorders remain unknown, and has yet to be studied in a single multi-disorder framework. Using MRI morphometry from the ENIGMA consortium, we construct maps of cortical abnormalities for thirteen neurodevelopmental, neurological, and psychiatric disorders from N = 21,000 participants and N = 26,000 controls, collected using a harmonised processing protocol. We systematically compare cortical maps to multiple micro-architectural measures, including gene expression, neurotransmitter density, metabolism, and myelination (molecular vulnerability), as well as global connectomic measures including number of connections, centrality, and connection diversity (connectomic vulnerability). We find a relationship between molecular vulnerability and white-matter architecture that drives cortical disorder profiles. Local attributes, particularly neurotransmitter receptor profiles, constitute the best predictors of both disorder-specific cortical morphology and cross-disorder similarity. Finally, we find that cross-disorder abnormalities are consistently subtended by a small subset of network epicentres in bilateral sensory-motor, inferior temporal lobe, precuneus, and superior parietal cortex. Collectively, our results highlight how local molecular attributes and global connectivity jointly shape cross-disorder cortical abnormalities.© 2022. The Author(s)."]},"columns":[{"id":".rownames","name":"","type":"character","sortable":false,"filterable":false,"rowHeader":true},{"id":"V1","name":"V1","type":"character","width":600}],"highlight":true,"outlined":true,"dataKey":"77648847f41cf86c2327a28dbcbecb3d","static":false,"nested":true},"children":[]},{"name":"Reactable","attribs":{"data":{".rownames":["Author","Abstract"],"V1":["Elman JA, Vogel JW, Bocancea DI, Ossenkoppele R, van Loenhoud AC, Tu XM, Kremen WS","Cognitive reserve and resilience are terms used to explain interindividual variability in maintenance of cognitive health in response to adverse factors, such as brain pathology in the context of aging or neurodegenerative disorders. There is substantial interest in identifying tractable substrates of resilience to potentially leverage this phenomenon into intervention strategies. One way of operationalizing cognitive resilience that has gained popularity is the residual method: regressing cognition on an adverse factor and using the residual as a measure of resilience. This method is attractive because it provides a statistical approach that is an intuitive match to the reserve/resilience conceptual framework. However, due to statistical properties of the regression equation, the residual approach has qualities that complicate its interpretation as an index of resilience and make it statistically inappropriate in certain circumstances.We describe statistical properties of the regression equation to illustrate why the residual is highly correlated with the cognitive score from which it was derived. Using both simulations and real data, we model common applications of the approach by creating a residual score (global cognition residualized for hippocampal volume) in individuals along the AD spectrum. We demonstrate that in most real-life scenarios, the residual measure of cognitive resilience is highly correlated with cognition, and the degree of this correlation depends on the initial relationship between the adverse factor and cognition. Subsequently, any association between this resilience metric and an external variable may actually be driven by cognition, rather than by an operationalized measure of resilience. We then assess several strategies proposed as potential solutions to this problem, such as including both the residual and original cognitive measure in a model. However, we conclude these solutions may be insufficient, and we instead recommend against \"pre-regression\" strategies altogether in favor of using statistical moderation (e.g., interactions) to quantify resilience.Caution should be taken in the use and interpretation of the residual-based method of cognitive resilience. Rather than identifying resilient individuals, we encourage building more complete models of cognition to better identify the specific adverse and protective factors that influence cognitive decline.© 2022. The Author(s)."]},"columns":[{"id":".rownames","name":"","type":"character","sortable":false,"filterable":false,"rowHeader":true},{"id":"V1","name":"V1","type":"character","width":600}],"highlight":true,"outlined":true,"dataKey":"067df9862fcfbfff7a17aebd2804aa11","static":false,"nested":true},"children":[]},{"name":"Reactable","attribs":{"data":{".rownames":["Author","Abstract"],"V1":["Vogel JW, Hansson O","The distribution of Alzheimer's disease (AD) tau pathology varies systematically and causes a diverse array of syndromes. This forum article provides a brief overview of key controversies in untangling the complexity of AD subtypes, explores potential causes of AD variability in the population, and discusses clinical relevance and future directions of research into AD heterogeneity.Copyright © 2022 The Authors. Published by Elsevier Ltd.. All rights reserved."]},"columns":[{"id":".rownames","name":"","type":"character","sortable":false,"filterable":false,"rowHeader":true},{"id":"V1","name":"V1","type":"character","width":600}],"highlight":true,"outlined":true,"dataKey":"e18dbb50335f41e4d8815dc03c95838f","static":false,"nested":true},"children":[]},{"name":"Reactable","attribs":{"data":{".rownames":["Author","Abstract"],"V1":["Levitis E, Vogel JW, Funck T, Hachinski V, Gauthier S, Vöglein J, Levin J, Gordon BA, Benzinger T, Iturria-Medina Y, Evans AC","Amyloid-beta deposition is one of the hallmark pathologies in both sporadic Alzheimer's disease and autosomal-dominant Alzheimer's disease, the latter of which is caused by mutations in genes involved in amyloid-beta processing. Despite amyloid-beta deposition being a centrepiece to both sporadic Alzheimer's disease and autosomal-dominant Alzheimer's disease, some differences between these Alzheimer's disease subtypes have been observed with respect to the spatial pattern of amyloid-beta. Previous work has shown that the spatial pattern of amyloid-beta in individuals spanning the sporadic Alzheimer's disease spectrum can be reproduced with high accuracy using an epidemic spreading model which simulates the diffusion of amyloid-beta across neuronal connections and is constrained by individual rates of amyloid-beta production and clearance. However, it has not been investigated whether amyloid-beta deposition in the rarer autosomal-dominant Alzheimer's disease can be modelled in the same way, and if so, how congruent the spreading patterns of amyloid-beta across sporadic Alzheimer's disease and autosomal-dominant Alzheimer's disease are. We leverage the epidemic spreading model as a data-driven approach to probe individual-level variation in the spreading patterns of amyloid-beta across three different large-scale imaging datasets (2 sporadic Alzheimer's disease, 1 autosomal-dominant Alzheimer's disease). We applied the epidemic spreading model separately to the Alzheimer's Disease Neuroimaging initiative (n = 737), the Open Access Series of Imaging Studies (n = 510) and the Dominantly Inherited Alzheimer's Network (n = 249), the latter two of which were processed using an identical pipeline. We assessed inter- and intra-individual model performance in each dataset separately and further identified the most likely subject-specific epicentre of amyloid-beta spread. Using epicentres defined in previous work in sporadic Alzheimer's disease, the epidemic spreading model provided moderate prediction of the regional pattern of amyloid-beta deposition across all three datasets. We further find that, whilst the most likely epicentre for most amyloid-beta-positive subjects overlaps with the default mode network, 13% of autosomal-dominant Alzheimer's disease individuals were best characterized by a striatal origin of amyloid-beta spread. These subjects were also distinguished by being younger than autosomal-dominant Alzheimer's disease subjects with a default mode network amyloid-beta origin, despite having a similar estimated age of symptom onset. Together, our results suggest that most autosomal-dominant Alzheimer's disease patients express amyloid-beta spreading patterns similar to those of sporadic Alzheimer's disease, but that there may be a subset of autosomal-dominant Alzheimer's disease patients with a separate, striatal phenotype.© The Author(s) 2022. Published by Oxford University Press on behalf of the Guarantors of Brain."]},"columns":[{"id":".rownames","name":"","type":"character","sortable":false,"filterable":false,"rowHeader":true},{"id":"V1","name":"V1","type":"character","width":600}],"highlight":true,"outlined":true,"dataKey":"36671814290e8ca54bf47d859d2810bb","static":false,"nested":true},"children":[]},{"name":"Reactable","attribs":{"data":{".rownames":["Author","Abstract"],"V1":["Berron D, Vogel JW, Insel PS, Pereira JB, Xie L, Wisse LEM, Yushkevich PA, Palmqvist S, Mattsson-Carlgren N, Stomrud E, Smith R, Strandberg O, Hansson O","In Alzheimer's disease, post-mortem studies have shown that the first cortical site where neurofibrillary tangles appear is the transentorhinal region, a subregion within the medial temporal lobe that largely overlaps with Brodmann area 35, and the entorhinal cortex. Here we used tau-PET imaging to investigate the sequence of tau pathology progression within the human medial temporal lobe and across regions in the posterior-medial system. Our objective was to study how medial temporal tau is related to functional connectivity, regional atrophy, and memory performance. We included 215 amyloid-β- cognitively unimpaired, 81 amyloid-β+ cognitively unimpaired and 87 amyloid-β+ individuals with mild cognitive impairment, who each underwent 18F-RO948 tau and 18F-flutemetamol amyloid PET imaging, structural T1-MRI and memory assessments as part of the Swedish BioFINDER-2 study. First, event-based modelling revealed that the entorhinal cortex and Brodmann area 35 show the earliest signs of tau accumulation followed by the anterior and posterior hippocampus, Brodmann area 36 and the parahippocampal cortex. In later stages, tau accumulation became abnormal in neocortical temporal and finally parietal brain regions. Second, in cognitively unimpaired individuals, increased tau load was related to local atrophy in the entorhinal cortex, Brodmann area 35 and the anterior hippocampus and tau load in several anterior medial temporal lobe subregions was associated with distant atrophy of the posterior hippocampus. Tau load, but not atrophy, in these regions was associated with lower memory performance. Further, tau-related reductions in functional connectivity in critical networks between the medial temporal lobe and regions in the posterior-medial system were associated with this early memory impairment. Finally, in patients with mild cognitive impairment, the association of tau load in the hippocampus with memory performance was partially mediated by posterior hippocampal atrophy. In summary, our findings highlight the progression of tau pathology across medial temporal lobe subregions and its disease stage-specific association with memory performance. While tau pathology might affect memory performance in cognitively unimpaired individuals via reduced functional connectivity in critical medial temporal lobe-cortical networks, memory impairment in mild cognitively impaired patients is associated with posterior hippocampal atrophy.© The Author(s) (2021). Published by Oxford University Press on behalf of the Guarantors of Brain."]},"columns":[{"id":".rownames","name":"","type":"character","sortable":false,"filterable":false,"rowHeader":true},{"id":"V1","name":"V1","type":"character","width":600}],"highlight":true,"outlined":true,"dataKey":"ebb86f1f444de62774c35270ca44b293","static":false,"nested":true},"children":[]},{"name":"Reactable","attribs":{"data":{".rownames":["Author","Abstract"],"V1":["Hansen JY, Markello RD, Vogel JW, Seidlitz J, Bzdok D, Misic B","Regulation of gene expression drives protein interactions that govern synaptic wiring and neuronal activity. The resulting coordinated activity among neuronal populations supports complex psychological processes, yet how gene expression shapes cognition and emotion remains unknown. Here, we directly bridge the microscale and macroscale by mapping gene expression patterns to functional activation patterns across the cortical sheet. Applying unsupervised learning to the Allen Human Brain Atlas and Neurosynth databases, we identify a ventromedial-dorsolateral gradient of gene assemblies that separate affective and perceptual domains. This topographic molecular-psychological signature reflects the hierarchical organization of the neocortex, including systematic variations in cell type, myeloarchitecture, laminar differentiation and intrinsic network affiliation. In addition, this molecular-psychological signature strengthens over neurodevelopment and can be replicated in two independent repositories. Collectively, our results reveal spatially covarying transcriptomic and cognitive architectures, highlighting the influence that molecular mechanisms exert on psychological processes.© 2021. The Author(s), under exclusive licence to Springer Nature Limited."]},"columns":[{"id":".rownames","name":"","type":"character","sortable":false,"filterable":false,"rowHeader":true},{"id":"V1","name":"V1","type":"character","width":600}],"highlight":true,"outlined":true,"dataKey":"ae7c0786ca7eb2ec167337541666511e","static":false,"nested":true},"children":[]},{"name":"Reactable","attribs":{"data":{".rownames":["Author","Abstract"],"V1":["Vogel JW, Young AL, Oxtoby NP, Smith R, Ossenkoppele R, Strandberg OT, La Joie R, Aksman LM, Grothe MJ, Iturria-Medina Y, Pontecorvo MJ, Devous MD, Rabinovici GD, Alexander DC, Lyoo CH, Evans AC, Hansson O","Alzheimer's disease (AD) is characterized by the spread of tau pathology throughout the cerebral cortex. This spreading pattern was thought to be fairly consistent across individuals, although recent work has demonstrated substantial variability in the population with AD. Using tau-positron emission tomography scans from 1,612 individuals, we identified 4 distinct spatiotemporal trajectories of tau pathology, ranging in prevalence from 18 to 33%. We replicated previously described limbic-predominant and medial temporal lobe-sparing patterns, while also discovering posterior and lateral temporal patterns resembling atypical clinical variants of AD. These 'subtypes' were stable during longitudinal follow-up and were replicated in a separate sample using a different radiotracer. The subtypes presented with distinct demographic and cognitive profiles and differing longitudinal outcomes. Additionally, network diffusion models implied that pathology originates and spreads through distinct corticolimbic networks in the different subtypes. Together, our results suggest that variation in tau pathology is common and systematic, perhaps warranting a re-examination of the notion of 'typical AD' and a revisiting of tau pathological staging."]},"columns":[{"id":".rownames","name":"","type":"character","sortable":false,"filterable":false,"rowHeader":true},{"id":"V1","name":"V1","type":"character","width":600}],"highlight":true,"outlined":true,"dataKey":"a58b2a3553743519351d68323fe51238","static":false,"nested":true},"children":[]},{"name":"Reactable","attribs":{"data":{".rownames":["Author","Abstract"],"V1":["Köbe T, Binette AP, Vogel JW, Meyer PF, Breitner JCS, Poirier J, Villeneuve S","Resting-state functional connectivity is suggested to be cross-sectionally associated with both vascular burden and Alzheimer's disease (AD) pathology. However, evidence is lacking regarding longitudinal changes in functional connectivity. This study includes 247 cognitively unimpaired individuals with a family history of sporadic AD (185 women/ 62 men; mean [SD] age of 63 [5.3] years). Plasma total-, HDL-, and LDL-cholesterol and systolic and diastolic blood pressure were measured at baseline. Global (whole-brain) brain functional connectivity and connectivity from canonical functional networks were computed from resting-state functional MRI obtained at baseline and ~3.5 years of annual follow-ups, using a predefined functional parcellation. A subsample underwent Aβ- and tau-PET (n=91). Linear mixed-effects models demonstrated that global functional connectivity increased over time across the entire sample. In contrast, higher total-cholesterol and LDL-cholesterol levels were associated with greater reduction of functional connectivity in the default-mode network over time. In addition, higher diastolic blood pressure was associated with global functional connectivity reduction. The associations were similar when the analyses were repeated using two other functional brain parcellations. Aβ and tau deposition in the brain were not associated with changes in functional connectivity over time in the subsample. These findings provide evidence that vascular burden is associated with a decrease in functional connectivity over time in older adults with elevated risk for AD. Future studies are needed to determine if the impact of vascular risk factors on functional brain changes precede the impact of AD pathology on functional brain changes.Copyright © 2021. Published by Elsevier Inc."]},"columns":[{"id":".rownames","name":"","type":"character","sortable":false,"filterable":false,"rowHeader":true},{"id":"V1","name":"V1","type":"character","width":600}],"highlight":true,"outlined":true,"dataKey":"edf6719b48b48da9ac832fed9ac7c187","static":false,"nested":true},"children":[]},{"name":"Reactable","attribs":{"data":{".rownames":["Author","Abstract"],"V1":["Young AL, Vogel JW, Aksman LM, Wijeratne PA, Eshaghi A, Oxtoby NP, Williams SCR, Alexander DC","Subtype and Stage Inference (SuStaIn) is an unsupervised learning algorithm that uniquely enables the identification of subgroups of individuals with distinct pseudo-temporal disease progression patterns from cross-sectional datasets. SuStaIn has been used to identify data-driven subgroups and perform patient stratification in neurodegenerative diseases and in lung diseases from continuous biomarker measurements predominantly obtained from imaging. However, the SuStaIn algorithm is not currently applicable to discrete ordinal data, such as visual ratings of images, neuropathological ratings, and clinical and neuropsychological test scores, restricting the applicability of SuStaIn to a narrower range of settings. Here we propose 'Ordinal SuStaIn', an ordinal version of the SuStaIn algorithm that uses a scored events model of disease progression to enable the application of SuStaIn to ordinal data. We demonstrate the validity of Ordinal SuStaIn by benchmarking the performance of the algorithm on simulated data. We further demonstrate that Ordinal SuStaIn out-performs the existing continuous version of SuStaIn (Z-score SuStaIn) on discrete scored data, providing much more accurate subtype progression patterns, better subtyping and staging of individuals, and accurate uncertainty estimates. We then apply Ordinal SuStaIn to six different sub-scales of the Clinical Dementia Rating scale (CDR) using data from the Alzheimer's disease Neuroimaging Initiative (ADNI) study to identify individuals with distinct patterns of functional decline. Using data from 819 ADNI1 participants we identified three distinct CDR subtype progression patterns, which were independently verified using data from 790 ADNI2 participants. Our results provide insight into patterns of decline in daily activities in Alzheimer's disease and a mechanism for stratifying individuals into groups with difficulties in different domains. Ordinal SuStaIn is broadly applicable across different types of ratings data, including visual ratings from imaging, neuropathological ratings and clinical or behavioural ratings data.Copyright © 2021 Young, Vogel, Aksman, Wijeratne, Eshaghi, Oxtoby, Williams and Alexander."]},"columns":[{"id":".rownames","name":"","type":"character","sortable":false,"filterable":false,"rowHeader":true},{"id":"V1","name":"V1","type":"character","width":600}],"highlight":true,"outlined":true,"dataKey":"b239762d40e2cfea064eb13d20f597e5","static":false,"nested":true},"children":[]},{"name":"Reactable","attribs":{"data":{".rownames":["Author","Abstract"],"V1":["Groot C, Yeo BTT, Vogel JW, Zhang X, Sun N, Mormino EC, Pijnenburg YAL, Miller BL, Rosen HJ, La Joie R, Barkhof F, Scheltens P, van der Flier WM, Rabinovici GD, Ossenkoppele R","To determine whether atrophy relates to phenotypical variants of posterior cortical atrophy (PCA) recently proposed in clinical criteria (i.e., dorsal, ventral, dominant-parietal, and caudal) we assessed associations between latent atrophy factors and cognition.We employed a data-driven Bayesian modeling framework based on latent Dirichlet allocation to identify latent atrophy factors in a multicenter cohort of 119 individuals with PCA (age 64 ± 7 years, 38% male, Mini-Mental State Examination 21 ± 5, 71% β-amyloid positive, 29% β-amyloid status unknown). The model uses standardized gray matter density images as input (adjusted for age, sex, intracranial volume, MRI scanner field strength, and whole-brain gray matter volume) and provides voxelwise probabilistic maps for a predetermined number of atrophy factors, allowing every individual to express each factor to a degree without a priori classification. Individual factor expressions were correlated to 4 PCA-specific cognitive domains (object perception, space perception, nonvisual/parietal functions, and primary visual processing) using general linear models.The model revealed 4 distinct yet partially overlapping atrophy factors: right-dorsal, right-ventral, left-ventral, and limbic. We found that object perception and primary visual processing were associated with atrophy that predominantly reflects the right-ventral factor. Furthermore, space perception was associated with atrophy that predominantly represents the right-dorsal and right-ventral factors. However, individual participant profiles revealed that the large majority expressed multiple atrophy factors and had mixed clinical profiles with impairments across multiple domains, rather than displaying a discrete clinical-radiologic phenotype.Our results indicate that specific brain behavior networks are vulnerable in PCA, but most individuals display a constellation of affected brain regions and symptoms, indicating that classification into 4 mutually exclusive variants is unlikely to be clinically useful.© 2020 American Academy of Neurology."]},"columns":[{"id":".rownames","name":"","type":"character","sortable":false,"filterable":false,"rowHeader":true},{"id":"V1","name":"V1","type":"character","width":600}],"highlight":true,"outlined":true,"dataKey":"10d96c347567570c3f171de2bbfb8cfd","static":false,"nested":true},"children":[]},{"name":"Reactable","attribs":{"data":{".rownames":["Author","Abstract"],"V1":["Vogel JW, Iturria-Medina Y, Strandberg OT, Smith R, Levitis E, Evans AC, Hansson O","Tau is a hallmark pathology of Alzheimer's disease, and animal models have suggested that tau spreads from cell to cell through neuronal connections, facilitated by β-amyloid (Aβ). We test this hypothesis in humans using an epidemic spreading model (ESM) to simulate tau spread, and compare these simulations to observed patterns measured using tau-PET in 312 individuals along Alzheimer's disease continuum. Up to 70% of the variance in the overall spatial pattern of tau can be explained by our model. Surprisingly, the ESM predicts the spatial patterns of tau irrespective of whether brain Aβ is present, but regions with greater Aβ burden show greater tau than predicted by connectivity patterns, suggesting a role of Aβ in accelerating tau spread. Altogether, our results provide evidence in humans that tau spreads through neuronal communication pathways even in normal aging, and that this process is accelerated by the presence of brain Aβ."]},"columns":[{"id":".rownames","name":"","type":"character","sortable":false,"filterable":false,"rowHeader":true},{"id":"V1","name":"V1","type":"character","width":600}],"highlight":true,"outlined":true,"dataKey":"973ec32f7f03a1c63696551416c246fb","static":false,"nested":true},"children":[]},{"name":"Reactable","attribs":{"data":{".rownames":["Author","Abstract"],"V1":["Vogel JW, La Joie R, Grothe MJ, Diaz-Papkovich A, Doyle A, Vachon-Presseau E, Lepage C, Vos de Wael R, Thomas RA, Iturria-Medina Y, Bernhardt B, Rabinovici GD, Evans AC","The functional organization of the hippocampus is distributed as a gradient along its longitudinal axis that explains its differential interaction with diverse brain systems. We show that the location of human tissue samples extracted along the longitudinal axis of the adult human hippocampus can be predicted within 2mm using the expression pattern of less than 100 genes. Futhermore, this model generalizes to an external set of tissue samples from prenatal human hippocampi. We examine variation in this specific gene expression pattern across the whole brain, finding a distinct anterioventral-posteriodorsal gradient. We find frontal and anterior temporal regions involved in social and motivational behaviors, and more functionally connected to the anterior hippocampus, to be clearly differentiated from posterior parieto-occipital regions involved in visuospatial cognition and more functionally connected to the posterior hippocampus. These findings place the human hippocampus at the interface of two major brain systems defined by a single molecular gradient."]},"columns":[{"id":".rownames","name":"","type":"character","sortable":false,"filterable":false,"rowHeader":true},{"id":"V1","name":"V1","type":"character","width":600}],"highlight":true,"outlined":true,"dataKey":"87f90af3400b85a030a89743a81280cf","static":false,"nested":true},"children":[]},{"name":"Reactable","attribs":{"data":{".rownames":["Author","Abstract"],"V1":["Pichet Binette A, Gonneaud J, Vogel JW, La Joie R, Rosa-Neto P, Collins DL, Poirier J, Breitner JCS, Villeneuve S, Vachon-Presseau E","Age being the main risk factor for Alzheimer's disease, it is particularly challenging to disentangle structural changes related to normal brain ageing from those specific to Alzheimer's disease. Most studies aiming to make this distinction focused on older adults only and on a priori anatomical regions. Drawing on a large, multi-cohort dataset ranging from young adults (n = 468; age range 18-35 years), to older adults with intact cognition (n = 431; age range 55-90 years) and with Alzheimer's disease (n = 50 with late mild cognitive impairment and 71 with Alzheimer's dementia, age range 56-88 years), we investigated grey matter organization and volume differences in ageing and Alzheimer's disease. Using independent component analysis on all participants' structural MRI, we first derived morphometric networks and extracted grey matter volume in each network. We also derived a measure of whole-brain grey matter pattern organization by correlating grey matter volume in all networks across all participants from the same cohort. We used logistic regressions and receiver operating characteristic analyses to evaluate how well grey matter volume in each network and whole-brain pattern could discriminate between ageing and Alzheimer's disease. Because increased heterogeneity is often reported as one of the main features characterizing brain ageing, we also evaluated interindividual heterogeneity within morphometric networks and across the whole-brain organization in ageing and Alzheimer's disease. Finally, to investigate the clinical validity of the different grey matter features, we evaluated whether grey matter volume or whole-brain pattern was related to clinical progression in cognitively normal older adults. Ageing and Alzheimer's disease contributed additive effects on grey matter volume in nearly all networks, except frontal lobe networks, where differences in grey matter were more specific to ageing. While no networks specifically discriminated Alzheimer's disease from ageing, heterogeneity in grey matter volumes across morphometric networks and in the whole-brain grey matter pattern characterized individuals with cognitive impairments. Preservation of the whole-brain grey matter pattern was also related to lower risk of developing cognitive impairment, more so than grey matter volume. These results suggest both ageing and Alzheimer's disease involve widespread atrophy, but that the clinical expression of Alzheimer's disease is uniquely associated with disruption of morphometric organization.© The Author(s) (2020). Published by Oxford University Press on behalf of the Guarantors of Brain. All rights reserved. For permissions, please email: journals.permissions@oup.com."]},"columns":[{"id":".rownames","name":"","type":"character","sortable":false,"filterable":false,"rowHeader":true},{"id":"V1","name":"V1","type":"character","width":600}],"highlight":true,"outlined":true,"dataKey":"0c98a0ff18b554fd3454c962a65a4646","static":false,"nested":true},"children":[]}]},{"id":"Title","name":"Title","type":"character","style":{"fontSize":"11px"},"width":500},{"id":"Auth","name":"Auth","type":"character","style":{"fontSize":"11px"}},{"id":"Journal","name":"Journal","type":"character","style":{"fontSize":"11px"}},{"id":"Year","name":"Year","type":"numeric","style":{"fontSize":"11px"},"width":80}],"filterable":true,"searchable":true,"showPageSizeOptions":true,"dataKey":"7b3bd1694160654b597b7acda6ece60d"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>



</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const disableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'prefetch';
    }
  }
  const enableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'stylesheet';
    }
  }
  const manageTransitions = (selector, allowTransitions) => {
    const els = window.document.querySelectorAll(selector);
    for (let i=0; i < els.length; i++) {
      const el = els[i];
      if (allowTransitions) {
        el.classList.remove('notransition');
      } else {
        el.classList.add('notransition');
      }
    }
  }
  const toggleGiscusIfUsed = (isAlternate, darkModeDefault) => {
    const baseTheme = document.querySelector('#giscus-base-theme')?.value ?? 'light';
    const alternateTheme = document.querySelector('#giscus-alt-theme')?.value ?? 'dark';
    let newTheme = '';
    if(darkModeDefault) {
      newTheme = isAlternate ? baseTheme : alternateTheme;
    } else {
      newTheme = isAlternate ? alternateTheme : baseTheme;
    }
    const changeGiscusTheme = () => {
      // From: https://github.com/giscus/giscus/issues/336
      const sendMessage = (message) => {
        const iframe = document.querySelector('iframe.giscus-frame');
        if (!iframe) return;
        iframe.contentWindow.postMessage({ giscus: message }, 'https://giscus.app');
      }
      sendMessage({
        setConfig: {
          theme: newTheme
        }
      });
    }
    const isGiscussLoaded = window.document.querySelector('iframe.giscus-frame') !== null;
    if (isGiscussLoaded) {
      changeGiscusTheme();
    }
  }
  const toggleColorMode = (alternate) => {
    // Switch the stylesheets
    const alternateStylesheets = window.document.querySelectorAll('link.quarto-color-scheme.quarto-color-alternate');
    manageTransitions('#quarto-margin-sidebar .nav-link', false);
    if (alternate) {
      enableStylesheet(alternateStylesheets);
      for (const sheetNode of alternateStylesheets) {
        if (sheetNode.id === "quarto-bootstrap") {
          toggleBodyColorMode(sheetNode);
        }
      }
    } else {
      disableStylesheet(alternateStylesheets);
      toggleBodyColorPrimary();
    }
    manageTransitions('#quarto-margin-sidebar .nav-link', true);
    // Switch the toggles
    const toggles = window.document.querySelectorAll('.quarto-color-scheme-toggle');
    for (let i=0; i < toggles.length; i++) {
      const toggle = toggles[i];
      if (toggle) {
        if (alternate) {
          toggle.classList.add("alternate");     
        } else {
          toggle.classList.remove("alternate");
        }
      }
    }
    // Hack to workaround the fact that safari doesn't
    // properly recolor the scrollbar when toggling (#1455)
    if (navigator.userAgent.indexOf('Safari') > 0 && navigator.userAgent.indexOf('Chrome') == -1) {
      manageTransitions("body", false);
      window.scrollTo(0, 1);
      setTimeout(() => {
        window.scrollTo(0, 0);
        manageTransitions("body", true);
      }, 40);  
    }
  }
  const isFileUrl = () => { 
    return window.location.protocol === 'file:';
  }
  const hasAlternateSentinel = () => {  
    let styleSentinel = getColorSchemeSentinel();
    if (styleSentinel !== null) {
      return styleSentinel === "alternate";
    } else {
      return false;
    }
  }
  const setStyleSentinel = (alternate) => {
    const value = alternate ? "alternate" : "default";
    if (!isFileUrl()) {
      window.localStorage.setItem("quarto-color-scheme", value);
    } else {
      localAlternateSentinel = value;
    }
  }
  const getColorSchemeSentinel = () => {
    if (!isFileUrl()) {
      const storageValue = window.localStorage.getItem("quarto-color-scheme");
      return storageValue != null ? storageValue : localAlternateSentinel;
    } else {
      return localAlternateSentinel;
    }
  }
  const darkModeDefault = false;
  let localAlternateSentinel = darkModeDefault ? 'alternate' : 'default';
  // Dark / light mode switch
  window.quartoToggleColorScheme = () => {
    // Read the current dark / light value 
    let toAlternate = !hasAlternateSentinel();
    toggleColorMode(toAlternate);
    setStyleSentinel(toAlternate);
    toggleGiscusIfUsed(toAlternate, darkModeDefault);
  };
  // Ensure there is a toggle, if there isn't float one in the top right
  if (window.document.querySelector('.quarto-color-scheme-toggle') === null) {
    const a = window.document.createElement('a');
    a.classList.add('top-right');
    a.classList.add('quarto-color-scheme-toggle');
    a.href = "";
    a.onclick = function() { try { window.quartoToggleColorScheme(); } catch {} return false; };
    const i = window.document.createElement("i");
    i.classList.add('bi');
    a.appendChild(i);
    window.document.body.appendChild(a);
  }
  // Switch to dark mode if need be
  if (hasAlternateSentinel()) {
    toggleColorMode(true);
  } else {
    toggleColorMode(false);
  }
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    // For code content inside modals, clipBoardJS needs to be initialized with a container option
    // TODO: Check when it could be a function (https://github.com/zenorocha/clipboard.js/issues/860)
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->




</body></html>